<!DOCTYPE html>
<html style="font-size: 16px;">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="description" content="">
    <meta name="page_type" content="np-template-header-footer-from-plugin">
    <title>ChIP-seq</title>
    <link rel="stylesheet" href="nicepage.css" media="screen">
<link rel="stylesheet" href="ChIP-seq.css" media="screen">
    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="nicepage.js" defer=""></script>
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    
    
    
    
    
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "",
		"url": "index.html",
		"logo": "images/0-Logo_from_FreeLogoDesign.png"
}</script>
    <meta property="og:title" content="ChIP-seq">
    <meta property="og:type" content="website">
    <meta name="theme-color" content="#478ac9">
    <link rel="canonical" href="index.html">
    <meta property="og:url" content="index.html">
  </head>
  <body class="u-body"><header class="u-clearfix u-header u-header" id="sec-9ffa"><div class="u-clearfix u-sheet u-sheet-1">
        <a href="" class="u-image u-logo u-image-1" data-image-width="183" data-image-height="83">
          <img src="images/0-Logo_from_FreeLogoDesign.png" class="u-logo-image u-logo-image-1" data-image-width="183">
        </a>
        <nav class="u-menu u-menu-dropdown u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><symbol id="menu-hamburger" viewBox="0 0 16 16" style="width: 16px; height: 16px;"><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</symbol>
</defs></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header> 
    <section class="u-clearfix u-valign-middle u-section-1" src="" id="sec-3f4f">
      <div class="u-expanded-height-lg u-expanded-height-md u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-ffba" style=""></use></svg>
        <svg class="u-svg-content" id="svg-ffba"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-clearfix u-layout-wrap u-layout-wrap-1">
        <div class="u-layout">
          <div class="u-layout-row">
            <div class="u-align-center u-container-style u-layout-cell u-left-cell u-size-25 u-layout-cell-1">
              <div class="u-container-layout u-container-layout-1">
                <div alt="" class="u-image u-image-circle u-image-1" data-image-width="1920" data-image-height="1280"></div>
              </div>
            </div>
            <div class="u-container-style u-layout-cell u-right-cell u-shape-rectangle u-size-35 u-layout-cell-2">
              <div class="u-container-layout u-container-layout-2">
                <h1 class="u-align-left u-custom-font u-text u-text-font u-title u-text-1">Chromatin-Immunoprecipitation Sequencing (ChIP-seq)</h1>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-2" id="carousel_889b">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Histones</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Before we begin
to discuss <i>Chromatin-Immunoprecipitation Sequencing</i> (ChIP-seq), we will
discuss <u>why</u> there is even a need to sequence the materials from a
chromatin. The strands of DNA present in each cell are not free-floating in the
nucleus, but are compacted tightly in a series of steps. The first key actor in
this compaction is the <i>histone</i> protein. Histones are large protein
complexes composed of two copies of four subunits, forming an octomer. The
first two subunits are Histone 2A (H2A) and Histone 2B (H2B), which form dimers
with each other. This gives us two copies of a larger subunit composed of both
H2A and H2B. Histone 4 (H4) and Histone 3 (H3) forms a tetrameric structure
with both of their copies, to give us one larger subunit. The H3-H4 tetramer
forms the core of the histone and combines with the two H2A-H2B dimers to form
the complete histone protein.&nbsp; &nbsp;<br>
                    <br>
                    <br>
                  </p>
                  <img src="images/11-ChipSeq-histone_Nature.png" alt="" class="u-image u-image-default u-image-1" data-image-width="1972" data-image-height="1288">
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-3" id="carousel_3f5b">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1" data-href="https://nicepage.com">
                <div class="u-container-layout u-container-layout-1">
                  <p class="u-align-justify u-text u-text-1">The histone
protein interacts with DNA to form a nucleosome by wrapping the DNA strands
around the histone. Simply put, histones are the proteins, while nucleosomes
are the protein-DNA complexes. The nucleosomes then compact further with the
help of Histone 1 (H1) in a stage-wise manner to form chromatin. The combination
of nucleosomes and H1 results in the <i>chromatosome</i>, which is looped in a
circular manner to form the <i>30nm fiber</i>. This 30nm fiber folds upon itself
in loops to form a strand that is the <i>300nm</i> fiber. The 300nm fiber are
compressed and bent to form the <i>250nm-wide fiber</i>. Notice here that the
description implies the radius of the fiber that it is being compacted into. Tight
coiling of the 250nm-wide fiber will produce a chromatid. Click on this <a href="https://www.youtube.com/watch?v=gbSIBhFwQ4s" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">link</a> for a visual
representation of the process. &nbsp;&nbsp;&nbsp;<br>
                    <br>
                  </p>
                  <img src="images/11-ChipSeq-DNAorganization_Nature.jpg" alt="" class="u-image u-image-default u-image-1" data-image-width="627" data-image-height="614">
                  <p class="u-align-justify u-text u-text-2">Although the
steps show a consistent type of histone protein or subunit used, there are
several variants of histone proteins. Some variants may be favored compared to others.
It is important to note here that the histone variants are not equivalent to
histone modifications! However, histone variants themselves have a regulatory
role during development. Histone modifications are post-translational modifications
(PTM) to the amino acid residues by adding or removing groups, which alters the
way DNA interacts with the histone, thereby affecting accessibility for other
proteins. There are several well-known modifications, such as methylation,
acetylation, and phosphorylation. Some are more recently discovered, such as
sumoylation and ubiquitination. The full list of modifications is long and can
be amended to in the following years as further research into PTMs are performed.
It is also important to note that these modifications are performed on the
amino acid residue of the histone and not DNA. Therefore, PTMs of histones are
non-DNA based modifiations that cause changes in chromatin patterns. This is the
primary focus in the field of epigenetics!&nbsp;&nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-4" id="carousel_9bb4">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1" data-href="https://nicepage.com">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">ChIP-seq</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">&nbsp; Now that we
know the significance of histones and chromatins, we can continue to understand
why ChIP-seq is performed. ChIP-seq is composed of two portions: the chromatin immunoprecipitation
(ChIP) and the sequencing (seq). First, we will start with the ChIP portion
with the immunoprecipitation process, which is a method of isolating, by
precipitation, a protein of interest using an antibody that specifically binds
to the protein. In addition to the specification of ChIP antibodies used, these
antibodies are also discriminant in targeting their targets. Antibodies used in
ChIP can be designed to target histone modifications like H3K4me3, which stands
for the tri-methylation (me3) of lysine 4 (K4) on histone 3 (H3). The result of
ChIP is the precipitated targets from the antibody that are composed of the
histone proteins and DNA bound to it. The second portion of ChIP-seq is to
sequence the DNA that were obtained from ChIP using NGS, with the reads from
NGS aligned to the reference genome. Regions with large overlaps of reads,
which is referred to as a “pileup” of reads, are interpreted as being
associated with the histone modification or histone of interest. However, there
are also background binding occurring in the process, as no biological action
is flawless. Therefore, negative controls must be used with ChIP-seq
experiments to ensure that the final result shows peaks that are significant
and not the “noise”.&nbsp; &nbsp;<br>
                    <br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-5" id="carousel_f1cf">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">ChIP-seq Data Analysis</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Similar to the
output from NGS runs, the raw reads from ChIP-seq are typically stored in a FASTQ
file. At times, but less common, it may be stored in FASTA format. There are
many publicly available ChIP-seq datasets published in in SRA and the Gene
Expression Omnibus (<a href="https://www.ncbi.nlm.nih.gov/geo/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">GEO</a>)
databases, both of which are hosted by NCBI. Obtaining the data may be slightly
tricky with the SRA database as it requires the <a href="https://hpc.nih.gov/apps/sratoolkit.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">SRA toolkit</a> to be
downloaded on a Unix server before using the command <b>fastq-dump</b>.
Alternatively, Galaxy has a tool to retrieve data from the SRA database through
the <i style="">Download and Extract Reads in FASTA/Q</i> tool.&nbsp; &nbsp;<br>
                    <br>After obtaining
the ChIP-seq data in FASTQ format, the same process described in the <a href="NextGenerationSequencing.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">previous chapter</a> can be used to trim, groom, and align
the reads to a reference genome. The difference in the pipeline for ChIP-seq
analysis is after the alignment is performed. The aligned sequences, either in
BAM or SAM file format, are used as inputs for <i>peak calling</i>. Peak
calling identifies the “peaks” in the reads that are aligned to a reference
genome and indicates which regions of DNA in the reference genome showed
association with the DNA from ChIP portion. In turn, the association indicates
that the protein of interest used in the ChIP portion is associated with this
region of DNA. The output of peak callers is usually in BED or WIG file formats
as they encompass multiple interval-spanning features. Unfortunately, there is
no “best” peak calling program, as the program that is used should be dependent
on a variety of factors. For instance, one may choose a certain program over
another based on how well the program can handle the following factors: the
presence of replicates, the performance of control experiments, the use of single-
or paired-ends, the type of protein that is immunoprecipitated, etc. Generally
speaking, common programs that are used include <i>MACS2 </i>and <i>Sicer</i>,
both of which are available on Galaxy. For a more extensive list of peak
calling programs, click <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4095679/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">here</a>.&nbsp; &nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-6" id="carousel_0649">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">ChIP-seq Controls</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">One of the factors for deciding upon a peak calling program is the type of control experiment utilized in the ChIP-seq experiment. ChIP-seq controls are important because they help distinguish between the noise in the experiment and the low, but meaningful, signals present in the sequenced reads. Consider the case where two reads align with a single mismatch. Is this an error from low-quality or incorrect bases during signaling? Is it a true signal, but present in a low proportion of the cells sampled? Is it just noise? These are questions that a proper control experiment can help resolve, which saves time from having to replicate the experiments that produced unusable data. Concurrently, another way to improve ChIP-seq signal detection is by sequencing and analyzing the control experiment.&nbsp;&nbsp;<br>&nbsp;<br>The choice of
what type of control experiment depends entire on what is being
immunoprecipitated and the overall conditions of the ChIP-seq experiment. There
are three major types of control experiments: DNA input control, IgG antibody
control, and untagged strains. The <i>DNA input control</i> method is generally
a good option for ChIP-seq as it recovers DNA that had not been immunoprecipitated,
which represents all the DNA available in the experiment. This can help
differentiate between what is a significant result and what is simply noise from
the experiment. The second, <i>IgG antibody control</i>, is not recommended for
ChIP-seq, but is recommended for other experiments. This control method performs
a mock ChIP experiment using an antibody that does not bind to nuclear
proteins, which results in small amounts of random DNA that are collected at the
end of the ChIP portion. However, the small amounts of DNA must be amplified
with PCR prior to sequencing, which adds more noise. Lastly, the <i>untagged strains</i> control method helps control for the antibody being used by using an antibody primed
to target a tag that is on the protein. Tags can be commonly used tags, such as
GFP. This method will also employ an untagged wildtype as the control to ensure
that any protein-DNA complex that is caught by the antibody in the untagged wildtype
would also appear in the experiment. However, the drawback to this method is
that it is not always possible to tag the protein.&nbsp;&nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-7" id="sec-7e4d">
      <div class="u-expanded-height u-flip-horizontal u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-3583"></use></svg>
        <svg class="u-svg-content" id="svg-3583"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-align-justify u-container-style u-group u-shape-rectangle u-white u-group-1">
        <div class="u-container-layout u-container-layout-1">
          <p class="u-text u-text-body-color u-text-1">Another goal of
ChIP-seq is to know how the protein binding to DNA can change under certain
conditions or in different cell/tissue types. This can be done with <i>differential
peak calling</i> programs that can compare the difference between two separate
sample sets to determine significance. These programs are designed specifically
for the purpose of comparing peaks between multiple sample sets that have
different conditions. For a full list of differential peak callers, click <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4095679/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">here</a>. Having
control samples and replicate experiments for this goal is important for the
comparison program. Another reason, too, is that some differential peak callers
require controls and/or replicates. View this sample workflow below on ChIP-seq analysis performed through Galaxy or visit <a href="https://usegalaxy.org/u/brian_poon/p/finalproject-chip-seq" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">this</a> page.
          </p>
        </div>
      </div>
      <div class="u-clearfix u-custom-html u-custom-html-1">
        <iframe src="https://usegalaxy.org/u/brian_poon/p/finalproject-chip-seq" description="Galaxy Page" height="525px" width="1030px"></iframe>
      </div>
    </section>
    <section class="u-clearfix u-section-8" id="sec-ea00">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-border-9 u-border-palette-1-light-2 u-container-style u-expanded-width u-group u-radius-50 u-shape-round u-group-1">
          <div class="u-container-layout u-container-layout-1">
            <h3 class="u-align-center u-text u-text-1">Chromatin-Immunoprecipitation Sequencing (ChIP-seq)</h3>
            <a href="RNA-seq.html" class="u-btn u-btn-round u-button-style u-hover-palette-4-base u-palette-1-base u-radius-2 u-btn-1">Next Unit&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 492.004 492.004" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M484.14,226.886L306.46,49.202c-5.072-5.072-11.832-7.856-19.04-7.856c-7.216,0-13.972,2.788-19.044,7.856l-16.132,16.136    c-5.068,5.064-7.86,11.828-7.86,19.04c0,7.208,2.792,14.2,7.86,19.264L355.9,207.526H26.58C11.732,207.526,0,219.15,0,234.002    v22.812c0,14.852,11.732,27.648,26.58,27.648h330.496L252.248,388.926c-5.068,5.072-7.86,11.652-7.86,18.864    c0,7.204,2.792,13.88,7.86,18.948l16.132,16.084c5.072,5.072,11.828,7.836,19.044,7.836c7.208,0,13.968-2.8,19.04-7.872    l177.68-177.68c5.084-5.088,7.88-11.88,7.86-19.1C492.02,238.762,489.228,231.966,484.14,226.886z"></path>
</g>
</g></svg><img></span>
            </a>
            <a href="NextGenerationSequencing.html" class="u-btn u-btn-round u-button-style u-hover-palette-1-light-2 u-palette-1-base u-radius-2 u-text-hover-palette-4-base u-btn-2">&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 447.243 447.243" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M420.361,192.229c-1.83-0.297-3.682-0.434-5.535-0.41H99.305l6.88-3.2c6.725-3.183,12.843-7.515,18.08-12.8l88.48-88.48    c11.653-11.124,13.611-29.019,4.64-42.4c-10.441-14.259-30.464-17.355-44.724-6.914c-1.152,0.844-2.247,1.764-3.276,2.754    l-160,160C-3.119,213.269-3.13,233.53,9.36,246.034c0.008,0.008,0.017,0.017,0.025,0.025l160,160    c12.514,12.479,32.775,12.451,45.255-0.063c0.982-0.985,1.899-2.033,2.745-3.137c8.971-13.381,7.013-31.276-4.64-42.4    l-88.32-88.64c-4.695-4.7-10.093-8.641-16-11.68l-9.6-4.32h314.24c16.347,0.607,30.689-10.812,33.76-26.88    C449.654,211.494,437.806,195.059,420.361,192.229z"></path>
</g>
</g></svg><img></span> Previous Unit
            </a>
          </div>
        </div>
      </div>
    </section>
    
    
       <footer class="u-align-center u-clearfix u-footer u-grey-80" id="sec-0012"><div class="u-clearfix u-sheet u-sheet-1"></div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
           
      <p class="u-text">This webpage was produced by Brian Poon as the Final Porfolio for class AS.410.610.82, &nbsp; <br>Bioinformatics: Tools for Genome Analysis</i>, at Johns Hopkins University.&nbsp;
      </p>
  </body>
</html>