<!DOCTYPE html>
<html style="font-size: 16px;">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="description" content="">
    <meta name="page_type" content="np-template-header-footer-from-plugin">
    <title>DatabasesGenomeBrowser</title>
    <link rel="stylesheet" href="nicepage.css" media="screen">
<link rel="stylesheet" href="DatabasesGenomeBrowser.css" media="screen">
    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="nicepage.js" defer=""></script>
	 <script src="https://cdn.jsdelivr.net/npm/igv@2.8.2/dist/igv.min.js"></script>
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    
    
   
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "",
		"url": "index.html",
		"logo": "images/0-Logo_from_FreeLogoDesign.png"
}</script>
    <meta property="og:title" content="DatabasesGenomeBrowser">
    <meta property="og:type" content="website">
    <meta name="theme-color" content="#478ac9">
    <link rel="canonical" href="index.html">
    <meta property="og:url" content="index.html">
  </head>
  <body class="u-body"><header class="u-clearfix u-header u-header" id="sec-9ffa"><div class="u-clearfix u-sheet u-sheet-1">
        <a href="Home.html" class="u-image u-logo u-image-1" data-image-width="183" data-image-height="83">
          <img src="images/0-Logo_from_FreeLogoDesign.png" class="u-logo-image u-logo-image-1" data-image-width="183">
        </a>
        <nav class="u-menu u-menu-dropdown u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><symbol id="menu-hamburger" viewBox="0 0 16 16" style="width: 16px; height: 16px;"><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</symbol>
</defs></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header> 
    <section class="u-clearfix u-valign-middle u-section-1" src="" id="sec-3f4f">
      <div class="u-expanded-height-lg u-expanded-height-md u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-ffba" style=""></use></svg>
        <svg class="u-svg-content" id="svg-ffba"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-clearfix u-layout-wrap u-layout-wrap-1">
        <div class="u-layout">
          <div class="u-layout-row">
            <div class="u-align-center u-container-style u-layout-cell u-left-cell u-size-25 u-layout-cell-1">
              <div class="u-container-layout u-container-layout-1">
                <div alt="" class="u-image u-image-circle u-image-1" data-image-width="1920" data-image-height="1280"></div>
              </div>
            </div>
            <div class="u-container-style u-layout-cell u-right-cell u-shape-rectangle u-size-35 u-layout-cell-2">
              <div class="u-container-layout u-container-layout-2">
                <h1 class="u-align-left u-custom-font u-text u-text-font u-title u-text-1">Databases<br>And<br>Genome Browsers
                </h1>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-2" id="carousel_889b">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">IGV</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">The genome browser, <i>Integrative Genomics
Viewer</i> (<a href="https://software.broadinstitute.org/software/igv/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">IGV</a>),
be used through the <a href="https://igv.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">web browser</a> or downloaded through
the Broad Institute. It is recommended it is downloaded as there are options
that are not available through the web version. The remarkability of IGV is
that it can expediently handle genomic-sized datasets, such as that from BAM or
SAM files (which will be discussed later), by adjusting the resolution based on
region browsing. For small regions, the resolution can be very high. For wide
regions, the resolution becomes low. This design came to be because IGV was
originally developed to meet the needs for large projects, such as the <i>
                      <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">Cancer
Genome Atlas</a></i> and <i>1000 Genomes Project</i>, to make data
visualization easy for all.&nbsp;<br>
                    <br>Some genome assemblies are already in present
in IGV, but additional genomes can be downloaded to the browser through a
server. The server option, which requires internet access, can be found under
the top menu bar at <i>Genomes</i> &gt; <i>Load Genome From Server</i>. Data
from external databases, which will be displayed in the form of tracks, can be
downloaded through <i>File</i> &gt; <i>Load From Server</i>. Other options to
load data externally through the internet include using URL links to specific databases.
Alternatively, local data files, such as BED and WIG files, can also be
uploaded to IGV and be viewed as tracks. Use the embedded IGV below!&nbsp; &nbsp;<br>
                  </p>
                  <div class="u-video u-video-1">
                    <div id="IGV-div" scale="tofit" onload= "igv.createBrowser(IGV-div, config)">
                  </div>
					  </div>
						<br>
						<br>
						<br>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-3" id="carousel_f1cf">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">UCSC Genome Browser</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">The database and genome browser, the <a href="https://genome.ucsc.edu/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">UCSC Genome Browser</a>, is primarily a
eukaryotic genome browser that consists of 97 eukaryotic genomes across 49
mammals. The UCSC Genome Browser, hosted by the University of California at
Santa Cruz, displays data (with external links to NCBI and Ensembl) through
tracks for information on mRNA, EST, RefSeq Genes, variations (SNPs), histone
modifications, gene expression, etc. The rate of update for the UCSC Genome
Browser will vary based on priorities, as primates and model organism genomes
are given higher priority.&nbsp;<br>
                    <br>Some tracks that can be found on the UCSC
Genome Browser includes the <i>UCSC </i>Genes, <i>Conservation</i>, and <i>PhastCons</i> tracks. The <i>UCSC Genes</i> track is a gene prediction set based on compiling
data from <i>
                      <a href="https://www.ncbi.nlm.nih.gov/refseq/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">RefSeq</a></i>, <i>GenBank</i>,
and <i>
                      <a href="https://www.uniprot.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">UniProt</a></i> databases. The <i>Conservation</i> track is displayed on the human and mouse genomes and shows conservation of the
genes in the region for up to 100 different species. The <i>PhastCons </i>track
uses the PhastCons program that has a two-state phylogenetic HMM, with one probabilistic
state for conserved regions and one probabilistic state for non-conserved regions,
to display the predicted conservation for the genes in the selected region. The
PhastCons program utilizes genome-wide alignments with limited species (human,
mouse, dog, zebrafish, and fly) from BLASTz and can be downloaded as <i>Multiple
Alignment Format </i>(<a href="http://www.bx.psu.edu/~dcking/man/maf.xhtml" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">MAF</a>)
files. These are just some examples as there are more tracks on the UCSC Genome
Browser for data on variation, gene expression, <a href="https://www.encodeproject.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">ENCODE</a>, etc. The UCSC Genome Browser
allows modification of tracks in display through five modes: <i>Hide</i>, <i>Dense</i>, <i>Full</i>, <i>Squish</i>, and <i>Pack</i>. It is of interest to note that the
use <i>Full</i> in track display should be done sparingly due to the large
number of track items displayed, and that the <i>Pack</i> option is usually
better-suited. In the UCSC Genome Browser, alternative versions of the genomic assembly
for an organism can be used. The Current default assembly for the human genome
is <i>GRCh38/hg38</i> from December 2013, which is updated for the gene and
mRNA tracks, but has some tracks that are not yet mapped to <i>hg38</i> from <i>hg19</i>.&nbsp;<br>
                    <br>Uniquely, UCSC Genome Browser has its own set
of tools that utilizes its database. We have seen before in a previous <a href="Eukaryotic-Genes.html" data-page-id="80703011" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-6">chapter</a> the NCBI’s alignment tool <i>Splign</i>. The UCSC
Genome Browser has its own alignment tool that is analogous to <i>Splign</i> and is called the <i>BLAST-like Alignment Tool</i> (<a href="https://www.encodeproject.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-7">BLAT</a>). However, as previously stated,
it does not have a graphical output and can align anywhere on the genome
without specifying a piece of genome DNA. Another tool is a data-mining program
called the <a href="https://genome.ucsc.edu/cgi-bin/hgTables?hgsid=1107199457_TDKFGWlDFmjBwz8rLNFqphmYJ3lR" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-8">UCSC
Table Browser</a>. This tool can retrieve data from the UCSC Genome Browser
database in a fashion similar to BioMart and biomaRt, but with a different set
of parameters and syntax. Please visit <a href="http://genome.ucsc.edu/goldenPath/help/hgTablesHelp.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-9">this</a> page
for a guide on how to use the Table Browser. The Table Browser also allows
uploading of files to use as a search parameter value, given that the
information in the file is in the correct format. The strongest benefit to the Table
Browser is its ability to transmit data directly to the Galaxy server, which
can be used to perform comparative genomic analysis. &nbsp;&nbsp;
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-4" id="carousel_0649">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Ensembl</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Similar in
nature to the UCSC Genome Browser, <a href="https://uswest.ensembl.org/info/about/index.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Ensembl</a> is genomic
database and browser hosted by the European Molecular Biology Laboratory's
European Bioinformatics Institute (<a href="http://www.ebi.ac.uk/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">EBI</a>) in
the United Kingdom, is primarily used for sequenced genomes of chordates. The
database includes non-chordate model organisms, but <u>not prokaryotes</u>. However,
the Ensembl database has non-vertebrate genomes through five subportals: Bacteria, <a href="https://protists.ensembl.org/index.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">Protist</a>, <a href="https://fungi.ensembl.org/index.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">Fungi</a>, <a href="http://plants.ensembl.org/index.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">Plants</a>, and <a href="https://metazoa.ensembl.org/index.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-6">Metazoa</a>. Archaeal genomes can
be found inside the bacteria subportal called <a href="https://bacteria.ensembl.org/index.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-7">EnsemblBacteria</a>. Ensembl is
a MySQL relational database that is updated approximately every four months. Users
can perform general searches based on genes or species-specific search. Click <a href="http://www.youtube.com/user/EnsemblHelpdesk" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-8">here</a> for a tutorial on navigating
the Ensembl database. Records in Ensembl use unique identifiers in the form of
11 digits with a heading based on the type of molecule and the species. For
example, the heading <i>ENSG</i> is used for human genes and <i>ENST</i> is
used for human transcripts. Ensembl was originally built for the human genome,
which is why headings for other species will be different. For example, the
heading for mouse genes is <i>ENSMUG</i>.&nbsp; &nbsp;<br>
                    <br>A general search in Ensembl can show general information on
chromosomes and the number of genes, further broken down as protein-coding or
non-coding. Clicking a gene will gather data from another database that
displays the gene’s information and specific regions in detail. On the
left-hand tab within the selected genes will display options for a variety of
information found for the gene, including transcripts, variants, protein
information, paralogs/orthologs, and external references. Clicking the link to
a transcript will display the transcript panel with similar options of deeper
information on the transcript. Transcript-specific information can also include
exonic and intronic information, as well as links to the cDNA and protein. Within
each panel is an embedded genome browser with tracks from GENCODE or other databases.
Furthermore, Ensembl allows customization options for users to customize the
dropdown menu.&nbsp; &nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-5" id="carousel_b81d">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Ensembl BioMart</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">
                    <a href="https://www.ensembl.org/biomart/martview/4e64d0e7d5ee0b888544cb9385ae2bce" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">BioMart</a> is a data-mining tool that utilizes information found through the Ensembl
Database by clicking the <i>BioMart</i> option on the top menu bar. BioMart
allows users to download data from Ensembl based on certain criteria used in
the search. First, select the <i>Database</i> to use. The primary database is <i>Ensembl
Genes</i>, but other databases include those specific for mouse strains,
variants, and regulatory data. The BioMart options can change based on the
version release, in which case if certain options are desired then archived version
of the database must first be accessed through <a href="http://mart.ensembl.org/info/website/archives/index.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">Ensembl
Archives</a> and BioMart selected. However, it is important to note that
although certain BioMart tools may be available in archived databases, those
archived databases do not contain the same up-to-date information.&nbsp;<br>&nbsp;&nbsp;&nbsp;<br>Within BioMart are subcategories listed under two main
categories: <i>Filters </i>and <i>Attributes</i>. The <i>Filters</i> category allow
you to select for genes that has certain features in the subcategories. Expand
the subcategories to find what subcategory you can search for: <i>Region</i>, <i>Gene</i>, <i>Phenotype</i>, <i>Gene Ontology</i>, <i>Multi Species comparisons</i>, <i>Protein Domains</i> and <i>Family</i>, and <i>Variant</i>. For example,
the <i>Region</i> subcategory can be used to pick out the physical region of
chromosome, locations, coordinates, etc. The <i>Gene</i> subcategory allows you
to pick out IDs, features with external databases, transcript counts, etc. The <i>Attributes</i> category allow you to display what you want to see by column in the result. The
selected subcategories under <i>Attributes</i> are the headings of the column
in the result. The top item under the <i>Attribute</i> link is the first column
and so on. There are three subcategories for the attributes: <i>Gene</i>, <i>External</i>, <i>Protein Domains</i>, and <i>Family</i>.&nbsp; &nbsp;<br>
                    <br>In summary, the result are the data are queried for based on
the <i>Filter </i>category, then displayed based on these columns with each
line down based on the <i>Attributes </i>category. &nbsp;This means that you can have 1 gene and 10
results because they’re all transcripts of that gene. Although each record is
unique the result table may not look unique if you don’t have all the
attributes showing. Within the resulting table on the webpage, the <i>Count </i>feature
at the top, next to <i>Results</i>, will show the number of genes found based
on the filter selections. The <i>Results</i> tab will only show a preview of 10
rows, regardless of the number of rows found. This can be adjusted by changing the
dropdown menu from <i>10</i> to <i>All</i>, which will open a separate webpage
with all the results. The results can also be exported. It is also important to
know that FASTA sequences can be exported with the results by using the radio
buttons on top under <i>Attributes</i>.&nbsp; &nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-6" id="carousel_370e">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Bioconductor and biomaRt in R</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Although it is not explicitly a database or genome browser, Bioconductor
and biomaRt is discussed here because they are data-mining tools that requires
a database to be used with it. Bioconductor is an open-source programming tool
in <i>
                      <a href="https://www.r-project.org/about.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">R</a></i> language and environment
that can be used to create customized pipelines to mine data from different
sources. One benefit of Bioconductor is that it is faster on the genome-scale when
compared to using genome browsers or maps. Within R is the <a href="https://bioconductor.org/packages/release/bioc/html/biomaRt.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">biomaRt</a> package, which is technically a Bioconductor package and are not considered
synonymous between each other. Bioconductor is the broader tool suite, while
biomaRt is the specific package within the suite. For information on how to set
up biomaRt in R, refer to this <a href="external/Unit%2004%20Reading%20-%20The%20biomaRt%20users%20guide_2018.pdf" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">guide</a> after R is installed. It is recommended to install <i>R Studio</i> rather than
the regular <i>R </i>installer for more options. To utilize biomaRt, the
Ensembl database and the specific dataset (based on the organism) must be
loaded first. For accessing the Ensembl database for <i>Homo sapiens </i>genes,
the command to do this in R is:&nbsp;<br><b>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp;<br>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; ensembl &lt;- useDataset(dataset =
"hsapiens_gene_ensembl", mart = ensembl)</b>&nbsp; &nbsp;<br>
                    <br>Once the database is loaded, use the <b>getBM()</b> command
in R to retrieve information. The concept of searching the database is identical
to that of Ensembl’s BioMart, which is a web-based version of the R-based
Bioconductor package. The <b>getBM()</b> command has four arguments: <i>attributes</i>, <i>filters</i>, <i>value</i>, and <i>mart</i>. The arguments will utilize the equal (=)
sign to indicate argument values. The <i>attributes</i> parameter is for the
columns to be displayed. The <i>filters</i> parameter contain the subcategories
to filter your query by, with the values of filtering based on the <i>value</i> parameter. The <i>mart</i> parameter is usually already established when selecting
the database and dataset. An example of a full command can be seen as:&nbsp; &nbsp;<br><b>
                      <br>getBM(attributes = c('affy_hg_u133_plus_2',
'entrezgene_id'), filters = 'affy_hg_u133_plus_2', values = c("202763_at","209310_s_at","207500_at"),
mart = ensembl)</b>&nbsp;<br>
                    <br>Note that the R-language allows the use of concatenation
commands, <b>c()</b>, within the parameter values, which will create a index-based
vector to link multiple values to the parameter. The example command above will
show two columns ('affy_hg_u133_plus_2' and 'entrezgene_id') after filtering
the database for the subcategory ('affy_hg_u133_plus_2') that contains the
values (‘202763_at’, ‘209310_s_at’, and ‘207500_at’) in the database (ensembl)
which was declared as a variable in the first sample command for <b>useDataset()</b>.
In creating variables, the results of the <b>getBM()</b> command can be stored
within another variable to be used for other R-based commands, such as <b>head()</b>.
For more information on R-language commands, please visit <a href="https://www.personality-project.org/r/r.commands.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">this</a> page. It is also important to note that the R-language does not require spacing
and delineates based on reserved characters. Aside from the <b>getBM() </b>command,
there are several others that can be used for accessing the database, such as <b>listAttributes()</b>, <b>listMart()</b>, etc. For a more thorough walkthrough on how to utilize
biomaRt commands, please visit <a href="https://bioconductor.org/packages/release/bioc/vignettes/biomaRt/inst/doc/accessing_ensembl.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">this</a> page for the documentation and <a href="https://www.youtube.com/watch?v=Rb4bl1zCv7I" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-6">this</a> page for a video guide.&nbsp; &nbsp;&nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-7" id="carousel_83b9">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Galaxy</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Although
Galaxy is also not explicitly a database or genome viewer, it is a tool that
heavily relies upon data uploaded from other databases, particular the UCSC Genome
Browser. <a href="https://usegalaxy.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Galaxy</a> is a web-based interface
that is used to analyze comparative genomic data retrieved from public
databases. The goals of comparative genomics are three-fold. The first is to
identify DNA sequences in a genome that are functional, as functions are
generally selected by evolution to be maintained. The second is to determine a
biological role of functional sequences by comparing the unknown sequences to
the known sequences, which is the concept behind the phylogenetic relationships
between orthologous and paralogous genes. The third goal is to compare the
evolutionary history of sequences through analysis of genomic data.&nbsp; &nbsp;<br>
                    <br>Galaxy,
itself, is an open-source framework that utilizes command-line (CML)
computation tools and database information to execute workflows. These workflows
are designed to be robust and flexible, while maintaining the reproducibility that
science requires. The user interface of the webpage is designed to allow users
without extensive programming knowledge to utilize bioinformatic tools in the
same manner. The Galaxy server is one of the best tools for starting
bioinformaticians that do not have a background in computer science and will
encompass an exercise in the next chapter! Galaxy has the ability to convert
data of different formats into the formats that are accepted and used by Galaxy.&nbsp; &nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-8" id="sec-7e4d">
      <div class="u-expanded-height u-flip-horizontal u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-3583"></use></svg>
        <svg class="u-svg-content" id="svg-3583"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-align-justify u-container-style u-group u-shape-rectangle u-white u-group-1">
        <div class="u-container-layout u-container-layout-1">
          <p class="u-text u-text-body-color u-text-1">The server itself is maintained, and contributed to, by groups of academics and individual users, who are collectively known as the <i>
              <a href="https://galaxyproject.org/community/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Galaxy community</a></i>. Galaxy can be used through the cloud server or it can be&nbsp;<a href="https://new.galaxyproject.org/admin/get-galaxy/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">installed locally</a>. The full potential of Galaxy is not able to be summed up in a single chapter, if not this entire tool, and will need to be learned through use by&nbsp;<a href="http://vimeo.com/channels/usegalaxy" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">tutorials</a>&nbsp;and&nbsp;<a href="https://galaxyproject.org/tutorials/g101/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">walkthroughs</a>. Please
see the embedded page below for a sample of Galaxy’s workflow pipeline or visit <a href="https://usegalaxy.org/u/brian_poon/p/finalproject" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">this </a> link here.
          </p>
        </div>
      </div>
      <div class="u-clearfix u-custom-html u-custom-html-1">
        <iframe src="https://usegalaxy.org/u/brian_poon/p/finalproject" description="Galaxy Page" height="525px" width="1030px"></iframe>
      </div>
    </section>
	 
	 <section class="u-clearfix u-section-9" id="sec-d940">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-border-9 u-border-palette-1-light-2 u-container-style u-expanded-width u-group u-radius-50 u-shape-round u-group-1">
          <div class="u-container-layout u-container-layout-1">
            <h3 class="u-align-center u-text u-text-1">Databases and Genome Browsers</h3>
            <a href="Genomic-File-Formats.html" class="u-btn u-btn-round u-button-style u-hover-palette-4-base u-palette-1-base u-radius-2 u-btn-1">Next Unit&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 492.004 492.004" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M484.14,226.886L306.46,49.202c-5.072-5.072-11.832-7.856-19.04-7.856c-7.216,0-13.972,2.788-19.044,7.856l-16.132,16.136    c-5.068,5.064-7.86,11.828-7.86,19.04c0,7.208,2.792,14.2,7.86,19.264L355.9,207.526H26.58C11.732,207.526,0,219.15,0,234.002    v22.812c0,14.852,11.732,27.648,26.58,27.648h330.496L252.248,388.926c-5.068,5.072-7.86,11.652-7.86,18.864    c0,7.204,2.792,13.88,7.86,18.948l16.132,16.084c5.072,5.072,11.828,7.836,19.044,7.836c7.208,0,13.968-2.8,19.04-7.872    l177.68-177.68c5.084-5.088,7.88-11.88,7.86-19.1C492.02,238.762,489.228,231.966,484.14,226.886z"></path>
</g>
</g></svg><img></span>
            </a>
            <a href="Variants.html" class="u-btn u-btn-round u-button-style u-hover-palette-1-light-2 u-palette-1-base u-radius-2 u-text-hover-palette-4-base u-btn-2">&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 447.243 447.243" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M420.361,192.229c-1.83-0.297-3.682-0.434-5.535-0.41H99.305l6.88-3.2c6.725-3.183,12.843-7.515,18.08-12.8l88.48-88.48    c11.653-11.124,13.611-29.019,4.64-42.4c-10.441-14.259-30.464-17.355-44.724-6.914c-1.152,0.844-2.247,1.764-3.276,2.754    l-160,160C-3.119,213.269-3.13,233.53,9.36,246.034c0.008,0.008,0.017,0.017,0.025,0.025l160,160    c12.514,12.479,32.775,12.451,45.255-0.063c0.982-0.985,1.899-2.033,2.745-3.137c8.971-13.381,7.013-31.276-4.64-42.4    l-88.32-88.64c-4.695-4.7-10.093-8.641-16-11.68l-9.6-4.32h314.24c16.347,0.607,30.689-10.812,33.76-26.88    C449.654,211.494,437.806,195.059,420.361,192.229z"></path>
</g>
</g></svg><img></span> &nbsp;Previous Unit
            </a>
          </div>
        </div>
      </div>
    </section>
	 
	 <script>
	 var igvDiv = document.getElementById("IGV-div");
      var options =
        {
            genome: "hg38",
            locus: "chr8:127,736,588-127,739,371",
            tracks: [
                {
                    "name": "HG00103",
                    "url": "https://s3.amazonaws.com/1000genomes/data/HG00103/alignment/HG00103.alt_bwamem_GRCh38DH.20150718.GBR.low_coverage.cram",
                    "indexURL": "https://s3.amazonaws.com/1000genomes/data/HG00103/alignment/HG00103.alt_bwamem_GRCh38DH.20150718.GBR.low_coverage.cram.crai",
                    "format": "cram"
                }
            ]
        };

        igv.createBrowser(igvDiv, options)
                .then(function (browser) {
                    console.log("Created IGV browser");
                })
	</script>
    
    
    <footer class="u-align-center u-clearfix u-footer u-grey-80" id="sec-0012"><div class="u-clearfix u-sheet u-sheet-1"></div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
           
      <p class="u-text">This webpage was produced by Brian Poon as the Final Porfolio for class AS.410.610.82, &nbsp; <br>Bioinformatics: Tools for Genome Analysis</i>, at Johns Hopkins University.&nbsp;
      </p>
      
    </section>
  </body>
</html>