<!DOCTYPE html>
<html style="font-size: 16px;">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="description" content="">
    <meta name="page_type" content="np-template-header-footer-from-plugin">
    <title>Eukaryotic Genes</title>
    <link rel="stylesheet" href="nicepage.css" media="screen">
<link rel="stylesheet" href="Eukaryotic-Genes.css" media="screen">
    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="nicepage.js" defer=""></script>
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    <link id="u-page-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    
    
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "",
		"url": "index.html",
		"logo": "images/0-Logo_from_FreeLogoDesign.png"
}</script>
    <meta property="og:title" content="Eukaryotic Genes">
    <meta property="og:type" content="website">
    <meta name="theme-color" content="#478ac9">
    <link rel="canonical" href="index.html">
    <meta property="og:url" content="index.html">
  </head>
  <body class="u-body"><header class="u-clearfix u-header u-header" id="sec-9ffa"><div class="u-clearfix u-sheet u-sheet-1">
        <a href="Home.html" class="u-image u-logo u-image-1" data-image-width="183" data-image-height="83">
          <img src="images/0-Logo_from_FreeLogoDesign.png" class="u-logo-image u-logo-image-1" data-image-width="183">
        </a>
        <nav class="u-menu u-menu-dropdown u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><symbol id="menu-hamburger" viewBox="0 0 16 16" style="width: 16px; height: 16px;"><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</symbol>
</defs></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header> 
    <section class="u-clearfix u-valign-middle u-section-1" src="" id="sec-3f4f">
      <div class="u-expanded-height-lg u-expanded-height-md u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-ffba" style=""></use></svg>
        <svg class="u-svg-content" id="svg-ffba"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-clearfix u-layout-wrap u-layout-wrap-1">
        <div class="u-layout">
          <div class="u-layout-row">
            <div class="u-align-center u-container-style u-layout-cell u-left-cell u-size-25 u-layout-cell-1">
              <div class="u-container-layout u-container-layout-1">
                <div alt="" class="u-image u-image-circle u-image-1" data-image-width="1920" data-image-height="1280"></div>
              </div>
            </div>
            <div class="u-container-style u-layout-cell u-right-cell u-shape-rectangle u-size-35 u-layout-cell-2">
              <div class="u-container-layout u-container-layout-2">
                <h1 class="u-align-left u-custom-font u-text u-text-font u-title u-text-1">Eukaryotic Genes<br>And<br>Prediction&nbsp;<br>
                </h1>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-2" id="carousel_889b">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Eukaryotic Gene Prediction</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">As
discussed before under the chapter <b>
                      <a href="Gene-Annotation-and-Structure.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Gene Annotation and Structure</a></b>, eukaryotic genes are monocistronic with the ability to produce
alternative protein products due to splicing. Although this layer of complexity
does add to the ability of predicting and annotating eukaryotic genes, many
programs for eukaryotic gene predictions are designed with this in mind. We
will see here that some of the gene prediction programs employ statistical
methods and sensors to increase accuracy. Eukaryotic gene prediction looks for
exons. The first exon begins with start codon because the 5’ UTR, 3’ UTR, and
non-coding exons are usually ignored by programs. In this case, the first
coding exon begins at the location where the start codon is adjacent to the
splice donor. Conversely, the last coding exon ends at where the stop codon is
adjacent to the splice acceptor.&nbsp;<br>
                    <br>Gene prediction
can have small-scale or large-scale goals. The approach largely depends on the
goal. An example of a small-scale goal can be finding or confirming the exonic
location of a gene. A large-scale goal can be to identify undiscovered genes. A
wholistic approach to predicting genes is by utilizing comparative genomics with
related species. The caveat here is that species must be relatively close, but distant.
For example, a chimpanzee genome would not be well-suited for comparing with a
human genome because the introns and non-coding DNA are almost as conserved as
the coding DNA. Likewise, a yeast genome is too distant from a human genome in
that the coding regions are not well conserved in evolutionary distance (as well
as the lack of yeast homologs). The middle ground would be to use a mice genome
that has the right level of conserved regions in the coding exons. Comparative
genomics can be used in addition to <i>de novo</i> prediction in order to improve
the prediction accuracy.&nbsp; &nbsp; &nbsp;&nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-3" id="carousel_f1cf">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1"><i>De Novo </i>Prediction Methods
                  </h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Eukaryotic gene predictions consist of two main types of methods
for prediction: <i>De novo</i> or expression-based. Expression-based methods
align the cDNA and protein sequences to genomic sequences to find the CDS, as
such the gene structures predicted by expression-based methods are very
accurate. However, the limitation is that expression-based methods require the
use of cDNA in the form of expressed sequence tags (ESTs) that may not be available
for certain genomes. Simultaneously, another limitation stems from the inherency
that 20% to 40% of most genomes code for rare genes, which makes generating ESTs
even harder. <i>De novo</i> (also known as <i>Ab initio</i>) gene prediction
programs analyze genomic sequences and predict from sequence information only
without referring to a genomic sequence. Programs for de novo gene prediction
include: <a href="http://hollywood.mit.edu/GENSCAN.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">GENSCAN</a>, <a href="http://www.cbs.dtu.dk/services/HMMgene/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">HMMGene</a>, <a href="http://www.softberry.com/berry.phtml?topic=fgenesh&amp;group=programs&amp;subgroup=gfind" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">FGENESH</a>,
and <a href="http://bioinf.uni-greifswald.de/augustus/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">Augustus</a>. GENSCAN,
developed at Standford and hosted at MIT, utilizes three categories of
probability in prediction of exons. The first category range starts from <i>very
high probability </i>(p &gt; 0.99), which are almost always correct and are
used with confidence for creating PCR primers. The second category range is <i>moderate
to high</i> (0.50 &lt; p &lt; 0.99), which are still quite accurate. The third
category range is <i>low probability</i> (p &lt; 0.50), which are not reliable
and should be ignored (or treated with caution). The benefits of GENSCAN
include a very organized output that includes FASTA-formatted sequences. However,
the predictions do not work very well with lower vertebrates, because it was
designed for human and vertebrae DNA. Interestingly, it does predict internal
exons better than the first and last exons, which is a normally expected outcome.
HMMGene, hosted by the Center for Biological Sequence Analysis (CBS), is a
prediction program that utilizes signal options for prediction of splice sites,
start codon, and stop codon. The program allows for a top-3 prediction rather
than a “best prediction”, which gives it a slower run time. HMMGene is designed
for human and <i>C. elegans</i> DNA, but is acceptable when used for all
vertebrates. Since <i>de novo</i> programs do not require ESTs, they succeed
where expression-based methods fail due to those limitations.&nbsp;&nbsp;
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-4" id="carousel_a829">
      <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
      <div class="u-clearfix u-gutter-30 u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-row">
            <div class="u-size-60">
              <div class="u-layout-col">
                <div class="u-align-justify u-container-style u-layout-cell u-right-cell u-size-60 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <h2 class="u-align-left u-text u-text-1">Expression-based Prediction Methods</h2>
                    <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-2"></div>
                    <p class="u-text u-text-2">Expression-based prediction methods align the sequenced cDNA to a known genome in order to extrapolate the CDS and UTRs, which are part of the cDNA. Programs for expression-based methods include&nbsp;<a href="http://www.ncbi.nlm.nih.gov/sutils/splign/splign.cgi?textpage=online&amp;level=form" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Splign</a>,&nbsp;<a href="http://www.ncbi.nlm.nih.gov/sutils/genom_table.cgi" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">genomic BLAST</a>, and the BLAST-like Alignment Tool (<a href="http://genome.ucsc.edu/cgi-bin/hgBlat?command=start" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">BLAT</a>). It is of interest to note that all three tools mentioned here are hosted by NCBI. Splign is used to align cDNA to a specific genomic sequence that must be added into the program. The&nbsp;<i>cDNA</i>&nbsp;portion is for the predicted or unknown sequence, while the&nbsp;<i>Genomic</i>&nbsp;portion is for the segment of reference genome. Genomic BLAST uses ESTs, which are essentially low-quality partial cDNA, to estimate the alignment to a reference genome. BLAT uses an input from the predicted or unknown sequence and aligns it to a reference genome assembly. It is similar to Splign, but vary in not having to provide the reference genome, which adds ease of use, and not having a graphical output.&nbsp; &nbsp;<br>
                    </p>
                    <div class="u-shape u-shape-svg u-text-palette-1-base u-shape-3">
                      <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 130" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-5496"></use></svg>
                      <svg class="u-svg-content" id="svg-5496" style="enable-background:new 0 0 160 130;"><path d="M148.3,0H11.7C5.3,0,0,5.2,0,11.5v79.8c0,6.4,5.3,11.5,11.7,11.5h56L67.6,130l24.2-27.2h56.5c6.5,0,11.7-5.2,11.7-11.5V11.5
	C160,5.2,154.7,0,148.3,0z"></path></svg>
                    </div>
                    <h6 class="u-text u-text-white u-text-3">
                      <span style="font-size: 1.5rem;"><b>&nbsp;Practice Time!&nbsp;<br></b>
                      </span>
                      <br>Go
to the link for Splign. Copy the sequence from <a href="external/znf214_mrna.txt" style="text-decoration:underline;" class="u-active-none u-border-none u-border-active-palette-2-dark-1 u-border-hover-palette-2-base u-btn u-button-link u-button-style u-hover-none u-text-body-alt-color u-text-hover-palette-2-base u-btn-5">this</a> file that
includes a FASTA format sequence the mRNA of ZNF214 from <i>Homo sapiens </i>into
the <i>cDNA </i>box. Copy the sequence from <a href="external/znf214_genomic.txt" style="text-decoration:underline;" class="u-active-none u-border-none u-border-active-palette-2-dark-1 u-border-hover-palette-2-base u-btn u-button-link u-button-style u-hover-none u-text-body-alt-color u-text-hover-palette-2-base u-btn-5">this</a> file that
includes a FASTA format of the genomic sequence for <i>Homo sapiens </i>box,
which includes the ZNF214 gene. Run Splign to obtain the mRNA and CDS
locations. Your result should look like below.<br>
                    </h6>
                    <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-4"></div>
                    <img src="images/5-EukaryoticGenesAndAnnotation-Splign_NCBI.png" alt="" class="u-image u-image-default u-image-1" data-image-width="1270" data-image-height="746">
                    <img src="images/5-EukaryoticGenesAndAnnotation-Splign2_NCBI.png" alt="" class="u-image u-image-default u-image-2" data-image-width="1272" data-image-height="358">
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-5" id="sec-7e4d">
      <div class="u-expanded-height u-flip-horizontal u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-3583"></use></svg>
        <svg class="u-svg-content" id="svg-3583"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-align-justify u-container-style u-group u-shape-rectangle u-white u-group-1">
        <div class="u-container-layout u-valign-top u-container-layout-1">
          <p class="u-text u-text-body-color u-text-1">Prediction
algorithms can contain additional tools that help improve its prediction
accuracy. One such tool is the inclusion of content sensor predictions that use
inherent features of the coding sequence, such as base composition. An example
of this is the bias against the third base in bacterial codon that are assumed
to be more random in non-coding DNA than coding DNA due to selective pressure.
One of the most reliable content sensors is hexamer frequency, which uses Hidden
Markov Models (HMMs) with codon bias based on the organism and di-codon
combinations. The HMM is generated from training sets used for determining
probability in using hexamer patterns through known genes to find unknown genes.
Another content sensor is a signal sensor program, which finds genes based on
sequence that is associated with the gene. Examples include signals for
promoters, splice sites, stop codons, and poly-A tails. Signal sensors find
signals using Position-Specific Scoring Matrices (PSSMs) that give more weight to
conserved regions than non-conserved regions. It is important to note that HMMs
can also be used to score signal sensors. A popular tool to employ signal and
content sensors is the use of neural networks with a training set, which is not
based on probability.&nbsp; &nbsp;</p>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-9" id="sec-d940">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-border-9 u-border-palette-1-light-2 u-container-style u-expanded-width u-group u-radius-50 u-shape-round u-group-1">
          <div class="u-container-layout u-container-layout-1">
            <h3 class="u-align-center u-text u-text-1">Eukaryotic Genes and Prediction</h3>
            <a href="RNA-and-Ultraconserved-Regions.html" class="u-btn u-btn-round u-button-style u-hover-palette-4-base u-palette-1-base u-radius-2 u-btn-1">Next Unit&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 492.004 492.004" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M484.14,226.886L306.46,49.202c-5.072-5.072-11.832-7.856-19.04-7.856c-7.216,0-13.972,2.788-19.044,7.856l-16.132,16.136    c-5.068,5.064-7.86,11.828-7.86,19.04c0,7.208,2.792,14.2,7.86,19.264L355.9,207.526H26.58C11.732,207.526,0,219.15,0,234.002    v22.812c0,14.852,11.732,27.648,26.58,27.648h330.496L252.248,388.926c-5.068,5.072-7.86,11.652-7.86,18.864    c0,7.204,2.792,13.88,7.86,18.948l16.132,16.084c5.072,5.072,11.828,7.836,19.044,7.836c7.208,0,13.968-2.8,19.04-7.872    l177.68-177.68c5.084-5.088,7.88-11.88,7.86-19.1C492.02,238.762,489.228,231.966,484.14,226.886z"></path>
</g>
</g></svg><img></span>
            </a>
            <a href="Prokaryotic-Genes.html" class="u-btn u-btn-round u-button-style u-hover-palette-1-light-2 u-palette-1-base u-radius-2 u-text-hover-palette-4-base u-btn-2">&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 447.243 447.243" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M420.361,192.229c-1.83-0.297-3.682-0.434-5.535-0.41H99.305l6.88-3.2c6.725-3.183,12.843-7.515,18.08-12.8l88.48-88.48    c11.653-11.124,13.611-29.019,4.64-42.4c-10.441-14.259-30.464-17.355-44.724-6.914c-1.152,0.844-2.247,1.764-3.276,2.754    l-160,160C-3.119,213.269-3.13,233.53,9.36,246.034c0.008,0.008,0.017,0.017,0.025,0.025l160,160    c12.514,12.479,32.775,12.451,45.255-0.063c0.982-0.985,1.899-2.033,2.745-3.137c8.971-13.381,7.013-31.276-4.64-42.4    l-88.32-88.64c-4.695-4.7-10.093-8.641-16-11.68l-9.6-4.32h314.24c16.347,0.607,30.689-10.812,33.76-26.88    C449.654,211.494,437.806,195.059,420.361,192.229z"></path>
</g>
</g></svg><img></span> &nbsp;Previous Unit
            </a>
          </div>
        </div>
      </div>
    </section>
    
    <footer class="u-align-center u-clearfix u-footer u-grey-80" id="sec-0012"><div class="u-clearfix u-sheet u-sheet-1"></div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
           
      <p class="u-text">This webpage was produced by Brian Poon as the Final Porfolio for class AS.410.610.82, &nbsp; <br>Bioinformatics: Tools for Genome Analysis</i>, at Johns Hopkins University.&nbsp;
      </p>
      
    </section>
  </body>
</html>