<!DOCTYPE html>
<html style="font-size: 16px;">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="description" content="">
    <meta name="page_type" content="np-template-header-footer-from-plugin">
    <title>Gene Annotation and Structure</title>
    <link rel="stylesheet" href="nicepage.css" media="screen">
<link rel="stylesheet" href="Gene-Annotation-and-Structure.css" media="screen">
    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="nicepage.js" defer=""></script>
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    <link id="u-page-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    
    
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "",
		"url": "index.html",
		"logo": "images/0-Logo_from_FreeLogoDesign.png"
}</script>
    <meta property="og:title" content="Gene Annotation and Structure">
    <meta property="og:type" content="website">
    <meta name="theme-color" content="#478ac9">
    <link rel="canonical" href="index.html">
    <meta property="og:url" content="index.html">
  </head>
  <body class="u-body"><header class="u-clearfix u-header u-header" id="sec-9ffa"><div class="u-clearfix u-sheet u-sheet-1">
        <a href="Home.html" class="u-image u-logo u-image-1" data-image-width="183" data-image-height="83">
          <img src="images/0-Logo_from_FreeLogoDesign.png" class="u-logo-image u-logo-image-1" data-image-width="183">
        </a>
        <nav class="u-menu u-menu-dropdown u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><symbol id="menu-hamburger" viewBox="0 0 16 16" style="width: 16px; height: 16px;"><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</symbol>
</defs></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header> 
    <section class="u-clearfix u-valign-middle u-section-1" src="" id="sec-3f4f">
      <div class="u-expanded-height-lg u-expanded-height-md u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-ffba" style=""></use></svg>
        <svg class="u-svg-content" id="svg-ffba"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-clearfix u-layout-wrap u-layout-wrap-1">
        <div class="u-layout">
          <div class="u-layout-row">
            <div class="u-align-center u-container-style u-layout-cell u-left-cell u-size-25 u-layout-cell-1">
              <div class="u-container-layout u-container-layout-1">
                <div alt="" class="u-image u-image-circle u-image-1" data-image-width="1920" data-image-height="1280"></div>
              </div>
            </div>
            <div class="u-container-style u-layout-cell u-right-cell u-shape-rectangle u-size-35 u-layout-cell-2">
              <div class="u-container-layout u-container-layout-2">
                <h1 class="u-align-left u-custom-font u-text u-text-font u-title u-text-1">Gene Annotation<br>And&nbsp;<br>Structure
                </h1>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-2" id="carousel_889b">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">GenBank Records&nbsp;</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">GenBank is a genetic sequence
database, hosted by the National Center for Biotechnology Information (NCBI),
that is part of the International Nucleotide Sequence Database Collaboration,
which includes the DNA DataBank of Japan (DDBJ) and the European Nucleotide
Archive (ENA)<a href="#_ftn1" name="_ftnref1" title="" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1"><!--[if !supportFootnotes]--><sup>[1]</sup><!--[endif]--></a>. GenBank
records provides submitted information on genes, transcripts, and proteins that
are tied to a unique <i>accession</i> number. Information can include a locus
identifier, definition, date of updated versions, molecular origin (mRNA, DNA, or
protein), length of the sequence, organism, comments, and publications. The
GenBank record can also provide raw sequences in FASTA format or link to a
web-based genome browser through the <i>Graphics </i>option. The most important
part of any GenBank record utilized by a bioinformatician is the features table.
The features table summarizes the protein or DNA record with annotations, or “features”,
of the transcript/sequence in a tabulated-format that includes sequence-specific
information and (if applicable) external references to outside databases. These
sequence-specific information can be segments of the sequence, such as introns,
exons, and the coding sequence (CDS). The CDS annotated is the region of DNA
that encodes the protein product. The CDS contains the coded exons only, in
which case there will be no 5’ UTR, 3’ UTR, or poly-A. Other features included in
the table are promoters, poly-A, repeats, and exons. Refer to <a href="http://www.insdc.org/documents/feature_table.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">here</a> for more information
on formats for annotations. Keep in mind the directionality of the strand
sequence. DNA is read from 5’ to 3’ direction on the plus strand, but sequences
can be found on the minus (or complement) strand. It will be indicated if the
sequence is on the minus strand with operators. The operator for complimentary
strand is “complement(&lt;location&gt;)”. Another operator is for joining
non-adjacent parts of sequences (i.e. when introns are spliced out). Refer to
3.4.2.2. of the documentation link referred earlier for more location
operators.&nbsp; <br><!--[if !supportFootnotes]-->
                    <br><!--[endif]-->
                    <br>
                    <a href="#_ftnref1" name="_ftn1" title="" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3"><!--[if !supportFootnotes]-->[1]<!--[endif]--></a> https://www.ncbi.nlm.nih.gov/genbank/&nbsp; &nbsp; &nbsp;&nbsp;
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-3" id="carousel_e5fd">
      <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
      <div class="u-clearfix u-gutter-30 u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-row">
            <div class="u-size-27 u-size-60-md">
              <div class="u-layout-col">
                <div class="u-align-left u-container-style u-layout-cell u-left-cell u-size-40 u-layout-cell-1" src="">
                  <div class="u-container-layout u-container-layout-1">
                    <img class="u-image u-image-1" src="images/3-GeneAnnotationsAndPredictions-genbank_NCBI.png" data-image-width="660" data-image-height="483">
                  </div>
                </div>
                <div class="u-align-justify u-container-style u-layout-cell u-left-cell u-size-20 u-layout-cell-2">
                  <div class="u-container-layout u-valign-top u-container-layout-2">
                    <img class="u-expanded-width u-image u-image-2" src="images/3-GeneAnnotationsAndPredictions-FeatureTable_NCBI.png" data-image-width="639" data-image-height="685">
                  </div>
                </div>
              </div>
            </div>
            <div class="u-size-33 u-size-60-md">
              <div class="u-layout-col">
                <div class="u-align-justify u-container-style u-layout-cell u-right-cell u-size-60 u-layout-cell-3">
                  <div class="u-container-layout u-container-layout-3">
                    <p class="u-text u-text-1">A challenge with GenBank record annotations is that only the original authors can submit updates to the GenBank record, unless expressed permission from the original authors were given to GenBank for the update. Further annotation updates to the record by a non-author can only be performed by sending experimental or inferential data to GenBank’s Third-Party Sequence Annotation (TPA) database. This poses a challenge in quality of records that results in GenBank being a redundant and non-curated database, which allows multiple submissions of sequences already present in the database. A non-author will be able to submit their data from their sequence even though the same sequence has a GenBank record. This redundancy also adds another layer to the lack of reliability and confidence. Albeit one can always defer to the newest update/submission as a method to filter through the noise, it is still the least ideal way to get the most updated record. As the database gets older, further questions may come to mind that: What if all the original authors were not able to be contacted? Take for example if they were all deceased. Would there be another submission record from another author be made for the original annotations to update any inaccurate annotations? Is there a de facto waiting period for contacting original authors before this is done? There is a beauty in redundancies, but in several decades (or even years) the database may become too overwhelmed with archaic data.&nbsp; &nbsp;<br>
                    </p>
                    <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-2"></div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-4" id="carousel_f1cf">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Annotation and Prediction</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Annotation
and Prediction
After sequencing, first focus is usually on prediction and
annotation of the sequence, especially with that of the CDS. Prediction is the
first step. Prediction often takes advantage of similarities between known
genes/proteins and models are made by aligning to known genes/proteins. Gene
prediction algorithms have been used in conjunction with automated annotation programs
to newly sequenced (full or partial) genomes. There are annotation pipeline
methods (i.e. MAKER and MAKER2) that use RNA-seq and other data for
annotations. Pipeline methods do make mistakes and finding errors are
difficult. There are tools (i.e. WebApollo) that help with reducing mistakes. Refer
to <a href="external/2012_Yandell_NatRev_annotation.pdf" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">this</a> article by Yandell and Ence (2012) on eukaryotic gene annotation for examples
of challenges with regards to splice variants.&nbsp;&nbsp;
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-5" id="sec-7e4d">
      <div class="u-expanded-height u-flip-horizontal u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-3583"></use></svg>
        <svg class="u-svg-content" id="svg-3583"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <p class="u-align-justify u-text u-text-1">A challenge with GenBank record annotations is that only the original authors can submit updates to the GenBank record, unless expressed permission from the original authors were given to GenBank for the update. Further annotation updates to the record by a non-author can only be performed by sending experimental or inferential data to GenBank’s Third-Party Sequence Annotation (TPA) database. This poses a challenge in quality of records that results in GenBank being a redundant and non-curated database, which allows multiple submissions of sequences already present in the database. A non-author will be able to submit their data from their sequence even though the same sequence has a GenBank record. This redundancy also adds another layer to the lack of reliability and confidence.&nbsp;<br>
        <br>
        <br>
      </p>
      <img src="images/3-GeneAnnotationsAndPredictions-annotationPipeline_NCBI.png" alt="" class="u-image u-image-default u-image-1" data-image-width="661" data-image-height="666">
      <div class="u-align-justify u-container-style u-group u-shape-rectangle u-white u-group-1">
        <div class="u-container-layout u-container-layout-1">
          <p class="u-text u-text-body-color u-text-default u-text-2">Albeit one can always defer to the newest update/submission as a method to filter? through the noise, it is still the least ideal way to get the most updated record. As the database gets older, further questions may come to mind that: What if all the original authors were not able to be contacted? Take for example if they were all deceased. Would there be another submission record from another author be made for the original annotations to update any inaccurate annotations? Is there a de facto waiting period for contacting original authors before this is done? There is a beauty in redundancies, but in several decades (or even years) the database may become too overwhelmed with archaic data.&nbsp; &nbsp;</p>
        </div>
      </div>
    </section>
    

	  <section class="u-clearfix u-section-9" id="sec-d940">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-border-9 u-border-palette-1-light-2 u-container-style u-expanded-width u-group u-radius-50 u-shape-round u-group-1">
          <div class="u-container-layout u-container-layout-1">
            <h3 class="u-align-center u-text u-text-1">Gene Annotation and Structure</h3>
            <a href="Prokaryotic-Genes.html" class="u-btn u-btn-round u-button-style u-hover-palette-4-base u-palette-1-base u-radius-2 u-btn-1">Next Unit&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 492.004 492.004" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M484.14,226.886L306.46,49.202c-5.072-5.072-11.832-7.856-19.04-7.856c-7.216,0-13.972,2.788-19.044,7.856l-16.132,16.136    c-5.068,5.064-7.86,11.828-7.86,19.04c0,7.208,2.792,14.2,7.86,19.264L355.9,207.526H26.58C11.732,207.526,0,219.15,0,234.002    v22.812c0,14.852,11.732,27.648,26.58,27.648h330.496L252.248,388.926c-5.068,5.072-7.86,11.652-7.86,18.864    c0,7.204,2.792,13.88,7.86,18.948l16.132,16.084c5.072,5.072,11.828,7.836,19.044,7.836c7.208,0,13.968-2.8,19.04-7.872    l177.68-177.68c5.084-5.088,7.88-11.88,7.86-19.1C492.02,238.762,489.228,231.966,484.14,226.886z"></path>
</g>
</g></svg><img></span>
            </a>
            <a href="Home.html" class="u-btn u-btn-round u-button-style u-hover-palette-1-light-2 u-palette-1-base u-radius-2 u-text-hover-palette-4-base u-btn-2">&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 447.243 447.243" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M420.361,192.229c-1.83-0.297-3.682-0.434-5.535-0.41H99.305l6.88-3.2c6.725-3.183,12.843-7.515,18.08-12.8l88.48-88.48    c11.653-11.124,13.611-29.019,4.64-42.4c-10.441-14.259-30.464-17.355-44.724-6.914c-1.152,0.844-2.247,1.764-3.276,2.754    l-160,160C-3.119,213.269-3.13,233.53,9.36,246.034c0.008,0.008,0.017,0.017,0.025,0.025l160,160    c12.514,12.479,32.775,12.451,45.255-0.063c0.982-0.985,1.899-2.033,2.745-3.137c8.971-13.381,7.013-31.276-4.64-42.4    l-88.32-88.64c-4.695-4.7-10.093-8.641-16-11.68l-9.6-4.32h314.24c16.347,0.607,30.689-10.812,33.76-26.88    C449.654,211.494,437.806,195.059,420.361,192.229z"></path>
</g>
</g></svg><img></span> Previous Unit
            </a>
          </div>
        </div>
      </div>
    </section>
	 
	 

     <footer class="u-align-center u-clearfix u-footer u-grey-80" id="sec-0012"><div class="u-clearfix u-sheet u-sheet-1"></div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
           
      <p class="u-text">This webpage was produced by Brian Poon as the Final Porfolio for class AS.410.610.82, &nbsp; <br>Bioinformatics: Tools for Genome Analysis</i>, at Johns Hopkins University.&nbsp;
      </p>
      
    </section>
    </section>
  </body>
</html>