<!DOCTYPE html>
<html style="font-size: 16px;">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="description" content="">
    <meta name="page_type" content="np-template-header-footer-from-plugin">
    <title>Genomic File Formats</title>
    <link rel="stylesheet" href="nicepage.css" media="screen">
<link rel="stylesheet" href="Genomic-File-Formats.css" media="screen">
    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="nicepage.js" defer=""></script>
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    <link id="u-page-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    
    
    
    
    
    
    
    
    
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "",
		"url": "index.html",
		"logo": "images/Logo1.png"
}</script>
    <meta property="og:title" content="Genomic File Formats">
    <meta property="og:type" content="website">
    <meta name="theme-color" content="#478ac9">
    <link rel="canonical" href="index.html">
    <meta property="og:url" content="index.html">
  </head>
  <body class="u-body"><header class="u-clearfix u-header u-header" id="sec-9ffa"><div class="u-clearfix u-sheet u-sheet-1">
        <a href="Home.html" class="u-image u-logo u-image-1" data-image-width="183" data-image-height="83">
          <img src="images/0-Logo_from_FreeLogoDesign.png" class="u-logo-image u-logo-image-1" data-image-width="183">
        </a>
        <nav class="u-menu u-menu-dropdown u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><symbol id="menu-hamburger" viewBox="0 0 16 16" style="width: 16px; height: 16px;"><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</symbol>
</defs></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header> 
    <section class="u-clearfix u-valign-middle u-section-1" src="" id="sec-3f4f">
      <div class="u-expanded-height-lg u-expanded-height-md u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-ffba" style=""></use></svg>
        <svg class="u-svg-content" id="svg-ffba"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-clearfix u-layout-wrap u-layout-wrap-1">
        <div class="u-layout">
          <div class="u-layout-row">
            <div class="u-align-center u-container-style u-layout-cell u-left-cell u-size-25 u-layout-cell-1">
              <div class="u-container-layout u-container-layout-1">
                <div alt="" class="u-image u-image-circle u-image-1" data-image-width="1920" data-image-height="1280"></div>
              </div>
            </div>
            <div class="u-container-style u-layout-cell u-right-cell u-shape-rectangle u-size-35 u-layout-cell-2">
              <div class="u-container-layout u-container-layout-2">
                <h1 class="u-align-left u-custom-font u-text u-text-font u-title u-text-1">Genomic File<br>Formats
                </h1>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-2" id="carousel_889b">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">BED file format</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">There are
numerous types of files used for storing and accessing genomic data. One of the
most common file types is the Browser Extensible Data (BED) file type. The BED
file is designed to store information on genomic intervals through tabulated
columns, such as a feature for an exon with the starting and ending position.
It has at the minimum three columns: the chromosome, position start, and
position end. Since BED files contain genomic positions, one of the first steps
of caution for a bioinformatician is to understand if the positions in the BED
file as based on a 0-based or 1-based interval. The difference between which interval
is used can affect the browser, albeit one will not know until it is compared
to a reference and the intervals are shifted by one position. For example, the
UCSC Genome Browser uses a 0-based interval. Large versions of the BED file
type (named BED-12) contains up to 12 columns, which include the strand, score
of the interval (relative to certain calculations), name, etc. The caveat here
is that columns go in sequential order based on the column type and cannot be
skipped. For example, you cannot go from column 3 to column 6 without column 4
and column 5. For more specific information on BED file formats, visit this <a href="https://genome.ucsc.edu/FAQ/FAQformat.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">link</a>, which also contains
information about other bioinformatic file types.&nbsp; &nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-3" id="carousel_6878">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">WIG and BigWig File Format</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Another common
file type is the <i>wiggle</i> (WIG) file type. The WIG file is designed to
store continuous data <i>at each</i> genomic location, such as chromosomal
modifications with locations, across a large genomic region. In the same as BED
files, WIG files can be used with genome browsers, but will require header
lines that contain metadata for the browser to interpret specific locations. An
alternate version of the WIG file format is the BigWig format, which is a
compressed version of the WIG format that omits non-relevant information for
genome browsers to load the display faster. While BIG and WIG, as well as BigWig, file formats are used for intervals on genomic locations, we will need something else to deal with larger amounts of data- such as that of an entire aligned, sequenced genome. The two most common file formats used are the&nbsp;<a href="https://samtools.github.io/hts-specs/SAMv1.pdf" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">SAM</a>&nbsp;and&nbsp;<a href="https://support.illumina.com/help/BS_App_RNASeq_Alignment_OLH_1000000006112/Content/Source/Informatics/BAM-Format.htm" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">BAM</a>&nbsp;files formats.&nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-4" id="carousel_f1cf">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">SAM File Format</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">The <i>Sequence Alignment/Map</i> (SAM) file format came to be
used during the 1000 Genomes Project. The challenge with the 1000 Genomes
Project is that it needs a file format to hold not only the sequence and
interval locations, but also the alignment of it to a reference genome after the
sequence is aligned. SAM file
formats are generally the result of aligning next-generation sequencing (NGS) from
ChIP-seq or RNA-seq, using alignment programs, such as HISAT2 or BWA. In
addition to location mapping, SAM file formats store information about how the
sequence aligns to the reference genome with 11 mandatory fields columns: QNAME, FLAG, RNAME,
POS, MAPQ, CIGAR, MRNM, MPOS, ISIZE, SEQ, and QUAL. Each field gives
information on the sequence in mapping to the reference genome. For example,
the <a href="https://genome.sph.umich.edu/wiki/SAM" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">CIGAR</a> field is a string
of symbols that encoding the variations in positions of the alignment between
the read sequence (the queried sequence) and the reference genome. Within the CIGAR
string, a <i>M</i> indicate that position was a match, a <i>I</i> indicate the
position had an insertion to the reference genome, and a <i>D</i> indicate the
position had a deletion from the reference genome. See below for a list of
fields and their description.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <img src="images/8-GenomicFileFormats-SAMfields_class.png" alt="" class="u-image u-image-default u-image-1" data-image-width="487" data-image-height="319">
      </div>
    </section>
    <section class="u-clearfix u-section-5" id="carousel_ec07">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-valign-bottom u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">BAM File Format</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Another common file
format for alignment results is the <i>Binary Alignment Map</i> (BAM) file
format. In actuality, a BAM file format contains the same information as the SAM
file format. However, the actual data within the BAM file format is compressed
and encoded into a binary form. The benefit of a BAM file is that compression
reduces the file size (in bytes) to be nearly equivalent to the number of base-pairs
in the sequenced file. The compression also allows for genomic browsers to retrieve
and display data at a faster rate, when it is sorted. On the topic of sorting, BAM
files need to be sorted in their intervals in relationship to the reference
genome. The <i>
                      <a href="https://depot.galaxyproject.org/hub/attachments/documents/papers/bamtools.pdf" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">BAMtools</a></i> suite in Galaxy automatically sorts BAM file outputs, but there is also the tool
command <i>sort</i> to sort BAM files if necessary. This step would be critical
when manipulating BAM files within a Unix shell. In addition to sorting, the
BAM files require a corresponding <i>BAM-Indexed</i> (BAI) file with the
same name when displaying the BAM file on a genomic browser. The BAI file
format is used by the browser, in the background, to correctly place the
intervals based on the indexing.&nbsp; &nbsp;<br>
                    <br>
                    <br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-6" id="carousel_ab10">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">BEDtools Suite</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">It is one thing to know the options of file formats- it is another to know how to manipulate file formats to obtain the needed data. We will begin with two popular suite of tools that manipulate data, both of which are found amongst Galaxy’s list of tools to utilize for bioinformaticians. The first suite of tools is the<i>
                      <a href="http://bedtools.readthedocs.org/en/latest/content/tools/intersect.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">BEDtools</a></i>&nbsp;suite, which is widely known as the “Swiss Army Knife” of manipulating genomic data for genomic intervals. Outside of Galaxy, the BEDtools suite can be installed on a&nbsp;<i>Unix</i>&nbsp;server and commands can be entered through a Unix shell to perform operations.&nbsp; &nbsp;<br>
                  </p>
                  <img src="images/8-GenomicFileFormats-bedtoolsOptions_class.png" alt="" class="u-image u-image-default u-image-1" data-image-width="1158" data-image-height="572">
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-7" id="carousel_a0e9">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <p class="u-align-justify u-text u-text-1">In addition to the figure below that contains a list of
commands for BEDtools, the command <b>bedtools -help</b> will also display
current command options for the installed version. The general syntax of the
commands on Unix will be performed in this manner: <b>bedtools &lt;command&gt;
[command options]</b>. Commands, such as <i>intersect</i>, <i>map</i>, or<i> sort</i>; can employ multiple options for the desired output. For example, the
command <i>intersect</i> is a tool used to find overlaps in the intervals of
features within different BED files. Using this example, the option <b>-wa</b> is
used to obtain the full interval of <i>file A</i> that overlaps with any part
of <i>file B</i>. The option <b>-v<i></i></b> is used to obtain intervals from <i>file
A</i> that does not intersect with any intervals on <i>file B</i>. Please see
the figure below for a graphical display of the <i>intersect</i> command’s
options.&nbsp;&nbsp;<br>
                  </p>
                  <img src="images/8-GenomicFileFormats-IntersectIntervals_class.png" alt="" class="u-image u-image-default u-image-1" data-image-width="853" data-image-height="350">
                  <br><br><p class="u-align-justify u-text u-text-2">Likewise, other commands can be used to get the desired
output. The figure below gives a broad overview of the commands from BEDtools
and what kind of output is given.&nbsp; &nbsp;<br>
                  </p>
                  <img src="images/8-GenomicFileFormats-bedtoolsGraphic_class.png" alt="" class="u-image u-image-default u-image-2" data-image-width="892" data-image-height="322">
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-8" id="carousel_b24b">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">SAMtools Suite</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">The second suite of tools is the <i>
                      <a href="http://www.htslib.org/download/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">SAMtools</a></i> suite, which is also
available on Galaxy or can be locally installed into a Unix server. The command <a href="http://www.htslib.org/doc/samtools.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">tools</a> within SAMtools
allow for manipulation of SAM file formats, as well as other alignment-based
file formats (BAM or CRAM). For example, the <i>view</i> command allows for
conversion of SAM file formats to BAM file formats. If performing operations
with SAMtools within a Unix shell the command is: <b>samtools view -bS
filename.sam &gt; filename.bam</b>. The <b>-b</b> option specifies the output
as a BAM file and the <b>-S</b> command specifies the input as a SAM file. SAMtools
also has the ability to perform operations on BAM files, such as the <i>sort</i> or <i>index</i> commands that, respectively, sort or index the BAM file
inputted to give the desired output. Note that the order of which options go
first is not necessary in Unix. For more information on performing commands
through a Unix shell, visit <a href="https://www.oreilly.com/library/view/learning-the-unix/1565923901/ch01s02.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">this</a> page for a broad overview of Unix command syntax. If using Galaxy, keep in mind
that, similar to BAMtools, there is not an explicit <i>index</i> command within
this suite as Galaxy automatically runs the <i>index</i> command as part of the
pipeline to generate the BAI file for other tools.&nbsp; &nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-9" id="carousel_a829">
      <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
      <div class="u-clearfix u-gutter-30 u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-row">
            <div class="u-size-60">
              <div class="u-layout-col">
                <div class="u-align-justify u-container-style u-layout-cell u-right-cell u-size-60 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <h2 class="u-align-left u-text u-text-1">Exercise Practice with BEDtools in Galaxy</h2>
                    <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-2"></div>
                    <p class="u-text u-text-2">For this practice, we will perform a longer exercise. The
tools you will need are two bed files and the Galaxy server. The <a href="external/hs_5q31_exons.bed" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">first</a> BED file contains
intervals of exons on the <i>q31</i> region of chromosome 5. The <a href="external/hs_5q31_cpg.bed" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">second</a> BED file
contains intervals of CpG regions in that same region on chromosome 5. Using the
Galaxy server, upload both files as a BED file type and leave all other fields
as default. Within Galaxy, find the <i>BEDtools Intersect Interval</i> tool (under
Genomic File Manipulation &gt; BED) on the left-hand column list. Alternatively,
you can also search for “BEDtools intersect interval”. Within BEDtools, you
will need to start with having the <i>File A</i> option selected for <i>hs_5q31_exons.bed</i> and the <i>File B</i> option selected for <i>hs_5q31_cpg.bed</i>. This means
that our file to use is the BED file with exon intervals and the file we are
comparing to is the file of CpG intervals. Now, we will answer the following four
questions:&nbsp; &nbsp;<br>
                    </p>
                    <div class="u-shape u-shape-svg u-text-palette-1-base u-shape-3">
                      <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 130" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-5496"></use></svg>
                      <svg class="u-svg-content" id="svg-5496" style="enable-background:new 0 0 160 130;"><path d="M148.3,0H11.7C5.3,0,0,5.2,0,11.5v79.8c0,6.4,5.3,11.5,11.7,11.5h56L67.6,130l24.2-27.2h56.5c6.5,0,11.7-5.2,11.7-11.5V11.5
	C160,5.2,154.7,0,148.3,0z"></path></svg>
                    </div>
                    <h6 class="u-text u-text-white u-text-3"><!--[if !supportLists]-->1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->How many unique exons intersect a CpG region?&nbsp;<!--[if !supportLists]-->
                      <br>
                      <br>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->How many unique exons do not intersect CpG
regions?&nbsp;<!--[if !supportLists]-->
                      <br>
                      <br>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->How many unique CpG regions intersect an exon?&nbsp;<!--[if !supportLists]-->
                      <br>
                      <br>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]-->How many unique CpG regions do not intersect
exons?&nbsp; &nbsp;<br>
                    </h6>
                    <p class="u-text u-text-4"><b>First Question:</b> For this part, the goal is to find
unique exons that intersect a CpG region. Thus, the (<b>-u</b>) option in the
parameters will be selected for to show only unique exons intersected. All
other parameters are held at default. The result showed that there were 542
unique exons that intersected a CpG region. Your result should be the same as
the figure below.&nbsp; &nbsp;<br>
                    </p>
                    <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-4"></div>
                    <img src="images/8-GenomicFileFormats-exercise1_Galaxy.png" alt="" class="u-image u-image-default u-image-1" data-image-width="284" data-image-height="344">
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-10" id="carousel_d736">
      <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
      <div class="u-clearfix u-gutter-30 u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-row">
            <div class="u-size-60">
              <div class="u-layout-col">
                <div class="u-align-justify u-container-style u-layout-cell u-right-cell u-size-60 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <p class="u-text u-text-1"><b>Second Question:</b> Using the same tool in a separate
instance, the option for (<b>-v</b>) will need to be selected for in order to
obtain exons that do not intersect with a CpG region. All other parameters are
held at default. The result showed that 4,296 exons did not intersect with a
CpG region. Your result should be the same as the figure below.&nbsp;&nbsp;<br>
                    </p>
                    <img src="images/8-GenomicFileFormats-exercise2_Galaxy.png" alt="" class="u-image u-image-default u-image-1" data-image-width="279" data-image-height="322">
                    <p class="u-text u-text-2"><b>Third Question:</b> Compared to the first and second
questions, we have <i>File A</i> set as the exon intervals file and <i>File B</i> set as the CpG intervals file. However, this question will not focus on the CpG
intervals file and compare that to the exon intervals file. Keep in mind that
the directionality does matter! In order to do this, we will need to set <i>File
A</i> for <i>hs_5q31_cpg.bed</i> and <i>File B </i>for <i>hs_5q31_exons.bed</i>.
Using the same tool in a separate instance, the (<b>-u</b>) option in the
parameters is selected for to show only unique CpG regions that intersected.
All other parameters are held at default. The result showed that there were 200
unique CpG regions that intersected an exon. Your result should be the same as
the figure below.&nbsp;&nbsp;<br>
                    </p>
                    <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-2"></div>
                    <img src="images/8-GenomicFileFormats-exercise3_Galaxy.png" alt="" class="u-image u-image-default u-image-2" data-image-width="265" data-image-height="333">
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-11" id="carousel_eb7c">
      <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
      <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-2"></div>
      <div class="u-clearfix u-gutter-30 u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-row">
            <div class="u-size-60">
              <div class="u-layout-col">
                <div class="u-align-justify u-container-style u-layout-cell u-right-cell u-size-60 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <p class="u-text u-text-1"><b>Fourth Question:</b> Set the file options in the
same manner as the third question. Using the same tool in a separate instance, the
(<b>-v</b>) option in the parameters is selected for to show only unique CpG
regions that did not intersect with an exon. All other parameters are held at
default. The result showed that there were 45 unique CpG regions that did not
intersect an exon. Your result should be the same as the figure below.&nbsp;&nbsp;<br>
                    </p>
                    <img src="images/8-GenomicFileFormats-exercise4_Galaxy.png" alt="" class="u-image u-image-default u-image-1" data-image-width="259" data-image-height="333">
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-12" id="sec-7e4d">
      <div class="u-expanded-height u-flip-horizontal u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-3583"></use></svg>
        <svg class="u-svg-content" id="svg-3583"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-align-justify u-container-style u-group u-shape-rectangle u-white u-group-1">
        <div class="u-container-layout u-container-layout-1">
          <h2 class="u-align-left u-text u-text-1"> Bonus Question! </h2>
          <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-2"></div>
          <p class="u-text u-text-body-color u-text-2">The Galaxy server is a highly malleable set of tools for any
bioinformatician to utilize. Best of all, the pipeline process in data
manipulation allows workflows to be saved and exported. This way your future
colleagues will be able to upload the workflow to view your parameter and
process, or run the same workflow. Just as it is a helpful to create workflows
for others, you can create one for yourself as reference or to save locally.
Galaxy is free to use, but the size is limited unless you create an account, which
extends your storage to 250GB of storage space. However, you will soon find out
that the limitations in storage space for processing bioinformatic data is a
constant struggle. In this bonus question, we will create a Galaxy workflow
based on the previous four questions. First, create an account and login. Be
sure to double check that your history from questions one to four are still
present in the history list on the right-hand column. That will be the history
list you will create a workflow for. First the history list panel (right-hand
column), click the cog-wheel icon for <i>History Options</i> and choose the
option for <i>Extract Workflow</i>. See below for a guide on the processes. Next,
create a name (as well as filename) for your workflow and ensure all the tools/data
in both columns are checked. Lastly, click <i>Create Workflow</i>. On the top
menu bar, click <i>Workflow</i> to open the page of your created workflows. You
also have the option of exporting the workflow as a <i>.ga</i> file format specific
for Galaxy. Click <a href="external/Part1.ga" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">here</a> for an example of the extracted workflow.&nbsp; &nbsp;
          </p>
        </div>
      </div>
      <img src="images/8-GenomicFileFormats-exercise5-2_Galaxy.png" alt="" class="u-image u-image-default u-image-1" data-image-width="528" data-image-height="632">
      <img src="images/8-GenomicFileFormats-exercise5-1_Galaxy.png" alt="" class="u-image u-image-default u-image-2" data-image-width="979" data-image-height="628">
    </section>
    
	 <section class="u-clearfix u-section-20" id="sec-d940">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-border-9 u-border-palette-1-light-2 u-container-style u-expanded-width u-group u-radius-50 u-shape-round u-group-1">
          <div class="u-container-layout u-container-layout-1">
            <h3 class="u-align-center u-text u-text-1">Genomic File Formats</h3>
            <a href="NextGenerationSequencing.html" class="u-btn u-btn-round u-button-style u-hover-palette-4-base u-palette-1-base u-radius-2 u-btn-1">Next Unit&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 492.004 492.004" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M484.14,226.886L306.46,49.202c-5.072-5.072-11.832-7.856-19.04-7.856c-7.216,0-13.972,2.788-19.044,7.856l-16.132,16.136    c-5.068,5.064-7.86,11.828-7.86,19.04c0,7.208,2.792,14.2,7.86,19.264L355.9,207.526H26.58C11.732,207.526,0,219.15,0,234.002    v22.812c0,14.852,11.732,27.648,26.58,27.648h330.496L252.248,388.926c-5.068,5.072-7.86,11.652-7.86,18.864    c0,7.204,2.792,13.88,7.86,18.948l16.132,16.084c5.072,5.072,11.828,7.836,19.044,7.836c7.208,0,13.968-2.8,19.04-7.872    l177.68-177.68c5.084-5.088,7.88-11.88,7.86-19.1C492.02,238.762,489.228,231.966,484.14,226.886z"></path>
</g>
</g></svg><img></span>
            </a>
            <a href="DatabasesGenomeBrowser.html" class="u-btn u-btn-round u-button-style u-hover-palette-1-light-2 u-palette-1-base u-radius-2 u-text-hover-palette-4-base u-btn-2">&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 447.243 447.243" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M420.361,192.229c-1.83-0.297-3.682-0.434-5.535-0.41H99.305l6.88-3.2c6.725-3.183,12.843-7.515,18.08-12.8l88.48-88.48    c11.653-11.124,13.611-29.019,4.64-42.4c-10.441-14.259-30.464-17.355-44.724-6.914c-1.152,0.844-2.247,1.764-3.276,2.754    l-160,160C-3.119,213.269-3.13,233.53,9.36,246.034c0.008,0.008,0.017,0.017,0.025,0.025l160,160    c12.514,12.479,32.775,12.451,45.255-0.063c0.982-0.985,1.899-2.033,2.745-3.137c8.971-13.381,7.013-31.276-4.64-42.4    l-88.32-88.64c-4.695-4.7-10.093-8.641-16-11.68l-9.6-4.32h314.24c16.347,0.607,30.689-10.812,33.76-26.88    C449.654,211.494,437.806,195.059,420.361,192.229z"></path>
</g>
</g></svg><img></span> &nbsp;Previous Unit
            </a>
          </div>
        </div>
      </div>
    </section>
	 
    
   <footer class="u-align-center u-clearfix u-footer u-grey-80" id="sec-0012"><div class="u-clearfix u-sheet u-sheet-1"></div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
           
      <p class="u-text">This webpage was produced by Brian Poon as the Final Porfolio for class AS.410.610.82, &nbsp; <br>Bioinformatics: Tools for Genome Analysis</i>, at Johns Hopkins University.&nbsp;
      </p>
      
    </section>
  </body>
</html>