<!DOCTYPE html>
<html style="font-size: 16px;">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="description" content="">
    <meta name="page_type" content="np-template-header-footer-from-plugin">
    <title>NextGenerationSequencing</title>
    <link rel="stylesheet" href="nicepage.css" media="screen">
<link rel="stylesheet" href="NextGenerationSequencing.css" media="screen">
    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="nicepage.js" defer=""></script>
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    <link id="u-page-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    
    
    
    
    
    
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "",
		"url": "index.html",
		"logo": "images/0-Logo_from_FreeLogoDesign.png"
}</script>
    <meta property="og:title" content="NextGenerationSequencing">
    <meta property="og:type" content="website">
    <meta name="theme-color" content="#478ac9">
    <link rel="canonical" href="index.html">
    <meta property="og:url" content="index.html">
  </head>
  <body class="u-body"><header class="u-clearfix u-header u-header" id="sec-9ffa"><div class="u-clearfix u-sheet u-sheet-1">
        <a href="Home.html" class="u-image u-logo u-image-1" data-image-width="183" data-image-height="83">
          <img src="images/0-Logo_from_FreeLogoDesign.png" class="u-logo-image u-logo-image-1" data-image-width="183">
        </a>
        <nav class="u-menu u-menu-dropdown u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><symbol id="menu-hamburger" viewBox="0 0 16 16" style="width: 16px; height: 16px;"><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</symbol>
</defs></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header> 
    <section class="u-clearfix u-valign-middle u-section-1" src="" id="sec-3f4f">
      <div class="u-expanded-height-lg u-expanded-height-md u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-ffba" style=""></use></svg>
        <svg class="u-svg-content" id="svg-ffba"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-clearfix u-layout-wrap u-layout-wrap-1">
        <div class="u-layout">
          <div class="u-layout-row">
            <div class="u-align-center u-container-style u-layout-cell u-left-cell u-size-25 u-layout-cell-1">
              <div class="u-container-layout u-container-layout-1">
                <div alt="" class="u-image u-image-circle u-image-1" data-image-width="1920" data-image-height="1280"></div>
              </div>
            </div>
            <div class="u-container-style u-layout-cell u-right-cell u-shape-rectangle u-size-35 u-layout-cell-2">
              <div class="u-container-layout u-container-layout-2">
                <h1 class="u-align-left u-custom-font u-text u-text-font u-title u-text-1">Next Generation Sequencing (NGS)&nbsp;<br>
                </h1>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-2" id="carousel_889b">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">NGS Sequencing Technologies</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">With the topic of Next Generation Sequencing
(NGS), we are about to come to a full circle from the introduction. This is
where the modern era of molecular biology, spurred by bioinformatics, is making
the debut. Within the last decade alone, advancements in NGS technology are
leading an explosion of sequenced data. However, as you might surmised from the
previous chapters, there is no “single” human genome- even though the current
(as of this writing) reference is <i>hg38</i>. The “reference” is generated
based on cumulative consensus of sequence data from individuals that determine,
based on population genetics, what the correct nucleotide should be. More
recently, sequenced data is more readily available through <i>whole exome
sequencing </i>(WES), which is an alterative to <i>whole genome sequencing </i>(WGS)
in that it only sequences the coding exons of an individual. The benefit of WES
is that only a portion of the individual’s genome is sequenced, which makes the
turnaround time faster and the cost lower.&nbsp;<br>
                    <br>The history of sequencing technologies begins
with first-generation sequencing: Sanger sequencing method, or also known as <i>chain
termination sequencing</i>. As the alternative name suggests, the method employs
unique nucleotides that terminates elongation at specific bases and sequences
are derived from overlapping these varied-length reads to determine the nucleotide
order. The entire process is labor-intensive and low-throughput. NGS, also
referred to as second-generation sequencing, is the next improvement from
Sanger sequencing. NGS technologies determine sequences based on detecting
nucleotides that were incorporated at each position during elongation. Examples
of NGS platforms include: Roche/454 Life Sciences (no longer in use), <a href="https://www.illumina.com/science/technology/next-generation-sequencing/sequencing-technology.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Illumina</a>, <a href="https://www.youtube.com/watch?v=nlvyF8bFDwM" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">ABI SOLiD</a>, <a href="https://www.youtube.com/watch?v=zBPKj0mMcDg" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">Ion Torrent</a>, <a href="http://www.pacb.com/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">Pacific Biosciences</a>, and <a href="https://www.youtube.com/watch?v=GUb1TZvMWsw&amp;t=17s" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">Oxford Nanopore</a>.
It is of interest to note that Oxford Nanopore is the latest technology that can
soon become a “third-generation” sequencing platform. For more information on
these sequencing technologies, visit <a href="https://www.youtube.com/watch?v=jFCD8Q6qSTM" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-6">this</a> page.&nbsp; &nbsp;&nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-3" id="carousel_f1cf">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-valign-top u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Phred Quality Scores and FASTQ File Format</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">As one can see, there are many platforms that
can produce sequences. However, the question now is how “good” are the
sequences produced? Each platform employs an encoding scheme specific to the platform
type. These encoding schemes utilize <a href="https://en.wikipedia.org/wiki/Phred_quality_score" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Phred quality scores</a> to determine the quality of the nucleotide called for the position in the read.
Phred scores grew from the Human Genome Project as a means of determining
confidence in the nucleotides that were called and were used to resolve discrepancies
with overlapping sequences. Sadly, the majority of original runs from the Human
Genome Project are nonexistent. The Sequence Read Archive (<a href="https://www.ncbi.nlm.nih.gov/sra" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">SRA</a>) is an international project
involving NCBI, EMBL, and DDBJ that aims to archive original, raw sequenced
data in the hopes of preserving original data. The SRA database contains publicly
available sequences in the original format for use. Unfortunately, the varying
encoding schemes still create discrepancies for how quality scores are encoded.
Nonetheless, the determination and analysis of Phred quality scores are still
widely used today as a critical component to the NGS analysis pipeline.&nbsp;<br>
                    <br>The
data produced by sequencers are held in a <i>FASTQ </i>format file type that includes
each nucleotide as single characters with an associated Phred quality score
represented by ASCII text codes that are positioned relative to the position of
the nucleotide in the read. Note here that “reads” are the sequences that result
from a sequencer and are small fragments of the entire genome. The FASTQ format
begins with header lines that indicate information for sequence reads or
quality scores, and will go in sequential order for the reads line and the
quality score line. The “@” symbol is the header for the reads and the “+”
symbol is the header for the quality score line. Within the FASTQ file, a set
will contain four header lines for each read: the header line for the read, the
read sequence, the header line for the quality score line, and the quality
score line.&nbsp; &nbsp;<br>
                    <br>One of the fields of study that NGS greatly
touched is the field of metagenomics, which is broadly defined as the study of
genetic material obtained from environmental samples. Alignment of the sequences
to a potential reference genome that match can indicate the presence of the
organism in the original environmental sample. This is commonly used in the
industry to rapidly detect the presence of unwanted pathogenic bacteria.&nbsp; &nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-4" id="carousel_f819">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Sequence Assembly</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">The
challenge after sequencing a sample is the task of properly assembling the
reads into full chromosomes, even if it is just a single strand. NGS can
massively sequence many small fragments in parallel, assemble the reads on
overlapping regions into contigs, and assemble contigs into full chromosomes.
Many programs have been designed to assemble fragments into genomes using the FASTQ
files as input after the FASTQ file has been trimmed to eliminate reads with
low-quality Phred scores.&nbsp;&nbsp;<br>
                    <br>For genomes that are sequenced for the very first
time in a particular species, there are programs that perform this <i>de novo</i> assembly. Once a reference genome becomes available, it serves as a guide for future
sequenced genomes. To a certain extent, reference genomes from a
closely-related organism may be used. However, the caveat here is that
evolutionarily it may still be distant enough to not be applicable. This
approach was somewhat success with prokaryotic species. Early genome assemblers
used a greedy algorithm to perform pairwise alignment among all fragments. Fragments
with most overlap are joined together to create longer sequences. This process
repeats until there are contigs. A huge drawback with using this method is that
repetitive regions cannot be resolved easily.&nbsp;<b></b>
                    <br>
                    <br>Sequencing can take two approaches: single-end
or paired-end. Single-end sequencing has each fragment sequenced from one end to
the other. Paired-end sequencing has both sides of each fragment sequenced simultaneously.
Although it is more tedious and lengthier, paired-end sequencing does provide
extra information on the reads, such as the original length of the fragment. The
importance here is to know which approach was used, as assembling programs will
need that information to properly assemble the reads.&nbsp;<b></b>
                    <br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-5" id="carousel_5cbc">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Reference-Based Assembly</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Mapping assemblers uses a reference genome to
assemble short reads to the reference genome.<b></b>This is useful for
assembling sequences from an individual human in researching cancer genomes. The
first step is to align trimmed FASTQ files to the reference genome<b>. </b>The
alignment program <i>Bowtie</i>, or <i>Bowtie2</i>, is perhaps the most
well-known alignment program. It produces acceptable with a fast speed using the
Burrows-Wheeler indexing approach described in 2009 by <a href="http://www.genomebiology.com/2009/10/3/R25" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Langmead et al</a>. Another popular
program is BWA, which also utilizes the Burrows-Wheeler indexing approach as
described in 2009 by <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">Li and Durbin</a>. While
either BWA or Bowtie can be used, the choice of either is generally dependent
on user preference. Lastly, there is a new program that utilizes the Burrows-Wheeler
indexing approach: HISAT. This program is soon becoming the new favorite with
its low memory usage and built-in functions to interface with downstream
analysis steps, as described in 2015 by <a href="https://www.nature.com/articles/nmeth.3317" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">Kim, Langmead, and Salzberg</a>.
All three alignment programs mentioned here are also available on Galaxy! Generally,
most reads will align perfectly to a reference genome, but there will be many reads
that do with a number of mismatches, with the number of mismatches depending on
the length of the read.&nbsp; &nbsp;<br>
                    <br>These mismatches are not always errors- there
are ways to troubleshoot for errors and perform quality control operations. Reads
may not match with the reference genome for various reasons. One of the reasons
is simply sequencing error- where the wrong base was called during sequencing. The
rate and types of sequencing error largely depends on the sequencing platform
that was used, with one or few reads found to contain errors when aligned to a
location on the reference genome. Another reason may be the presence of
heterozygous SNPs, which are sequences that match at a location on some reads,
but not in others. This is due to the presence of the SNP on one chromosome and
not on the other. Conversely, the presence of homozygous SNPs will have sequences
mismatch at a specific location entirely, because both chromosomes have the
SNP. These mismatches are reflective of SNPs and variants that are naturally
present in the individual. Some technician-related errors include PCR amplification
errors. Errors introduced early on to the reads during PCR is a single error,
but will continue to propagate throughout the process and produce multiple
copies of the error. This creates a problem for variant calling programs (discussed
later) when determining variants. Generally, the “deeper” the library is sequenced,
the more reads will be produced and the greater the coverage for the location
will be. This can help determine if the mismatches are variants or if they are true
errors. For example, if a heterozygous SNP is present, then it will show approximately
50% of the reads having a mismatch and approximately 50% of the reads having
aligned for that location. With the same concept, homozygous SNPs will show all
reads mismatching at that SNP’s location during alignment.&nbsp; &nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-6" id="carousel_69b8">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Variant Calling</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">The practical use of sequencing individual
genomes is to analyze the data. One type of analysis that is commonly performed
is <i>variant calling</i>. As the name suggests, variant calling is the method
of determining, based on statistical analysis, if the mismatch is a SNP or if
it is an error. With previously discussed on deep sequencing libraries, more
reads often equates to more confidence in variant calling. Some examples of
variant calling programs include: <i>
                      <a href="https://github.com/ekg/freebayes" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">FreeBayes</a></i>, <i>
                      <a href="http://www.htslib.org/doc/samtools.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">mpileup</a></i> from <i>SAMtools</i>, <i>
                      <a href="here" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">GATK Unified Genotyper</a></i>, <i>
                      <a href="https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeCaller.php" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">GATK
Haplotype Caller</a></i>, and <i>
                      <a href="http://www.well.ox.ac.uk/platypus" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">Platypus</a></i>.
Variant callers are used after the alignment program is finished. The BAM file,
containing the aligned reads to a reference genome, outputted from the alignment
program will become the input to variant callers in the analysis pipeline. The
output of variant callers is typically a <i>Variant Call Format</i> (<a href="https://samtools.github.io/hts-specs/VCFv4.2.pdf" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-6">VCF</a>) file, with
each variant called represented as a line in the VCF file. This is almost analogous
to the BED file format, but they are not the same. Also, most programs will
output the number of reads aligned at the position of the variant called.&nbsp; &nbsp;<br>
                    <br>Variant calling is an analysis pipeline on
its own, beginning with FASTQ files from NGS and ending with a VCF file of
called variants. There are different programs that can achieve this pipeline, but
we will discuss an example pipeline. In our example, the first step in the pipeline
is to use the FASTQ file with <i>FASTQ Groomer</i> tool to reformat the Phred
score encoding scheme to <i>Sanger/Illumina 1.9</i> encoding. This is needed in
order to ensure that our Phred scores were encoded to a scheme that the next
tool is able to use. The next step is to trim the groomed FASTQ file with
either <i>FASTQ Trimmer</i> or <i>Trimmomatic</i> in order to remove all the reads
(or regions, depending on specific parameters) that have low-quality scores. This
will set us up with success in the alignment. In between step one and step two
is a critical step of checking the quality of the reads using the <i>FASTQC</i> tool. Once acceptable, the third step is to align the groomed and trimmed FASTQ
file to a reference genome using either <i>Bowtie2</i>, <i>BWA</i>, or <i>HISAT2</i>.
If necessary, the fourth step is to convert from a SAM to BAM file. If
necessary, or needed, the fifth step is to filter the BAM file to a genomic
region, which will save time. The sixth step, if necessary, is to sort the BAM
file. The seventh step is to use the aligned BAM file to call variants using a
variant caller, such as <i>FreeBayes</i>. After variant calling, the variants
can be assessed with annotation information to predict how the variants affect
gene or protein function, which is also available through Galaxy.&nbsp; &nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-7" id="sec-7e4d">
      <div class="u-expanded-height u-flip-horizontal u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-3583"></use></svg>
        <svg class="u-svg-content" id="svg-3583"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-align-justify u-container-style u-group u-shape-rectangle u-white u-group-1">
        <div class="u-container-layout u-valign-top u-container-layout-1">
          <p class="u-text u-text-body-color u-text-1">Below, you can find the general steps. Note
that all these tools are available in Galaxy, which will reduce some steps as
Galaxy has automated certain tools to be used in the output.&nbsp;<b></b><!--[if !supportLists]-->
            <br>
            <br>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]--><b>FASTQ Groomer:</b> Reformat the FASTQ files
to Sanger/Illumina 1.9&nbsp;<br>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]--><b>FASTQ Trimmer / Trimmomatic:</b> Trim away
regions with low quality scores<b></b>&nbsp;<br>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]--><b>Bowtie2 / BWA / HISAT:</b> Align the reads to
a reference genome<b></b>&nbsp;<br>4.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]--><b>SAM-to-BAM</b>: Convert the output from the alignment
if necessary. This is not needed if using Galaxy.&nbsp;<br>5.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]--><b>Filter BAM</b>: Limit the BAM file to a
chromosome or genomic region to save time. This is optional.&nbsp;<br>6.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]--><b>Sort BAM:</b> This is optional, but is not
needed if using Galaxy.<b></b>&nbsp;<br>7.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <!--[endif]--><b>FreeBayes:</b> Use the aligned BAM file to
call for variants!<b></b>&nbsp; &nbsp; &nbsp;
          </p>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-8" id="carousel_a829">
      <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
      <div class="u-clearfix u-gutter-30 u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-row">
            <div class="u-size-60">
              <div class="u-layout-col">
                <div class="u-align-justify u-container-style u-layout-cell u-right-cell u-size-60 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <h2 class="u-align-left u-text u-text-1">Exercise Practice!</h2>
                    <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-2"></div>
                    <p class="u-text u-text-2">For
this practice, we will be attempting to call variants using tools from Galaxy
and NGS data that is publicly available on the 1000 Genomes Project. <u>Keep in
mind</u> that this raw data uses the human reference genome hg19, which is not
the most up to date reference assembly. However, for the purpose of
instruction, we need to maintain the same reference genome when it is
specified. You will be starting out with two raw, FASTQ files that represent a
paired-end sequencing experiment. The forward reads will have the file name
ending in “_1”. The reverse reads will have the file name ending in “_2”. Upload
both files into Galaxy with the <i>Paste/Fetch
Data</i> option and the FTP links given below. Be sure to set the datatype to <i>FASTQ</i> and the genome to <i>hg19</i>.&nbsp;<br>
                      <br><b>Forward:&nbsp;</b>ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/data/HG00117/sequence_read/SRR044234_1.filt.fastq.gz <b>Reverse:</b>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/phase3/data/HG00117/sequence_read/SRR044234_2.filt.fastq.gz<br>
                      <br>
                      <br><b>Step 1: </b>Perform <i>FASTQC</i> (under <i>Genomic File Manipulation</i> &gt; <i>FASTQ Quality Control</i>) on both files to
ensure that the <i>Encoding</i> result is <i>Sanger/Illumina 1.9</i>. If it is not, use
the tool <i>FASTQ Groomer</i> (under <i>Genomic File Manipulation</i> &gt; <i>FASTA/FASTQ</i>) and convert the file to <i>Sanger/Illumina 1.9</i>, which is the
default output. If it is, use the <i>Edit</i> option on the file (within the <i>History</i> panel) to change the file type to <i>fastqsanger</i>.&nbsp;<br>
                      <br><b>Step
2:</b> We will trim both
files with either <i>FASTQ Quality Trimmer</i> (under <i>Genomic File Manipulation</i> &gt; <i>FASTA/FASTQ</i>) or <i>Trimmomatic </i>(under<i> Genomic
File Manipulation </i>&gt;<i> FASTQ Quality
Control</i>).
For this exercise, we will use <i>Trimmomatic</i>.
Select <i>Paired-end</i> as the option on
the top, and choose the forward read FASTQ file for <i>R1</i> and reverse read FASTQ file for <i>R2</i>. Ensure that the <i>Sliding
window trimming</i> option is selected under <i>Trimmomatic operation</i>. Set the <i>number
of bases to average across</i> as <i>4</i> and the <i>average quality required</i> as
20. Execute <i>Trimmomatic</i>. There will
be four files as the resulting output. For the next steps, be sure to <u>only</u> use the trimmed files from the <i>paired</i> output. As an added assurance, perform <i>FASTQC</i> on the trimmed files to ensure that they are trimmed. You can confirm this by
viewing the <i>Per base quality sequence
boxplot</i> in the output. See below for a comparison of before (left) and
after (right) trimming.<br>
                      <br>
                    </p>
                    <img src="images/10-NGS-forwardUntrim_galaxy.png" alt="" class="u-image u-image-default u-image-1" data-image-width="700" data-image-height="796">
                    <img src="images/10-NGS-forwardTrim_galaxy.png" alt="" class="u-image u-image-default u-image-2" data-image-width="680" data-image-height="787">
                    <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-3"></div>
                    <p class="u-text u-text-3"><b>Step
3: </b>Align the reads to the reference genome (hg19) using <i>BWA</i>, <i>Bowtie2</i>,
or <i>HISAT2</i>. Both <i>BWA </i>and <i>Bowtie2</i> can be
found under <i>Common Genomics Tools</i> &gt; <i>Mapping</i>. Only <i>HISAT2</i> will be found under <i>Common Genomics Tools</i> &gt; <i>RNA-seq</i>. The choice would not affect the
result, but the options for <i>paired-end</i> must be selected for! In this example, we will use <i>Bowtie2</i>. Select <i>paired-end</i> on the top option. Use the trimmed, forward read FASTQ file for the <i>FASTA/Q file #1</i>. Use the trimmed,
reverse read FASTQ file for the <i>FASTA/Q
file #2</i> parameter. Under <i>Select
reference genome</i>, find and select for <i>Human
(Homo sapiens) (b37): hg19</i>. Keep all other parameters at default and
execute <i>Bowtie2</i>. The resulting output
will be a BAM file.&nbsp;<br>
                      <br><b>Step
4: </b>Use the tool <i>FreeBayes</i> (under <i>Genomics Analysis</i> &gt; <i>Variant Calling</i>) to identify variants.
Select the post-alignment output as the <i>BAM
dataset</i> and use <i>hg19</i> as the
reference genome. All other parameters can be set at default, but in this
exercise we will limit the output to only the region chr22:0-51304566 to save
time. Under the option for <i>limit variant
calling to a set of regions</i>, select the <i>limit
to region</i> option. Use <i>chr22</i> as
the <i>region chromosome</i>. Use <i>0</i> as the <i>region start</i>. Use <i>51304566</i> as the region end. Execute <i>FreeBayes</i>.
The output will be a VCF file of all variants. Compare your file to the file <a href="external/Part4_SRR044234_1_2_bowtie2_freebayes.vcf" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">here</a>.
Technically, this is where variant calling can end as they are the variants
called. However, we can still take it a step further by filtering and
annotating variants.&nbsp;<br>
                      <br><b>Step 5a: </b>While this step is optional, we will use the <i>VCFfilter</i> (under <i>Genomic File
Manipulation</i> &gt; <i>VCF/BCF</i>) tool
to filter for variants that show heterozygosity, in which we will expect an
estimated allele frequency of 0.5, and have more than 10 reads coverage, which
means the <u>depth</u> is greater than 10. The filtered attributes are
estimated allele frequency (AF) and read depth (DP). The AF and DP identifiers
can be found in the VCF file under the ##INFO lines. These filtered values will
be <i>AF=0.5</i> and <i>DP&gt;10</i>. In <i>VCFfilter</i>,
two filters were added with both listed as <i>-f </i>for Info filter in the <i>Select filter
type</i> category. In the <i>Specify
filtering value </i>category, the first filter had <i>AF = 0.5 </i>and the second filter had <i>DP &gt; 10</i>. Note the spaces were added to prevent errors! Execute <i>VCFfilter</i> with all other parameters were
held at default. Compare your result with <a href="external/Part4_SRR044234_1_2_bowtie2_freebayes_filtered.vcf" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">this</a> result to confirm that only six variants were found.&nbsp;<br>
                      <br><b>Step
5b</b>: We can use the <i>VCFannotate </i>(under <i>Genomic File Manipulation</i> &gt; <i>VCF/BCF</i>) tool to annotate the called
variants with the genes by intersecting the filtered VCF file from <i>Step 5a</i> to <a href="external/Part4_RefSeq_chr22.bed" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">this</a> BED file that contains all the
genes on chromosome 22, which was exported from the <i>RefSeq</i> track using the UCSC Table Browser. In the options, select
the BED from the link above and the filtered VCF file from <i>Step 5a</i>. Execute <i>VCFannotate</i> with all other parameters at default to obtain the final product. Compare your
result with <a href="external/Part4_SRR044234_1_2_bowtie2_freebayes_filtered_annotated.vcf" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">this</a> file to confirm that the features were annotated with <i>BED-features</i> tags.&nbsp;<br>
                      <br>Take a look at <a href="external/Part4b.ga" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">this</a> Galaxy workflow file and compare it to your history/workflow!&nbsp;&nbsp;<br>
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-9" id="sec-d940">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-border-9 u-border-palette-1-light-2 u-container-style u-expanded-width u-group u-radius-50 u-shape-round u-group-1">
          <div class="u-container-layout u-container-layout-1">
            <h3 class="u-align-center u-text u-text-1">Next Generation Sequencing (NGS)</h3>
            <a href="ChIP-seq.html" data-page-id="41865537" class="u-btn u-btn-round u-button-style u-hover-palette-4-base u-palette-1-base u-radius-2 u-btn-1">Next Unit&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 492.004 492.004" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M484.14,226.886L306.46,49.202c-5.072-5.072-11.832-7.856-19.04-7.856c-7.216,0-13.972,2.788-19.044,7.856l-16.132,16.136    c-5.068,5.064-7.86,11.828-7.86,19.04c0,7.208,2.792,14.2,7.86,19.264L355.9,207.526H26.58C11.732,207.526,0,219.15,0,234.002    v22.812c0,14.852,11.732,27.648,26.58,27.648h330.496L252.248,388.926c-5.068,5.072-7.86,11.652-7.86,18.864    c0,7.204,2.792,13.88,7.86,18.948l16.132,16.084c5.072,5.072,11.828,7.836,19.044,7.836c7.208,0,13.968-2.8,19.04-7.872    l177.68-177.68c5.084-5.088,7.88-11.88,7.86-19.1C492.02,238.762,489.228,231.966,484.14,226.886z"></path>
</g>
</g></svg><img></span>
            </a>
            <a href="Genomic-File-Formats.html" class="u-btn u-btn-round u-button-style u-hover-palette-1-light-2 u-palette-1-base u-radius-2 u-text-hover-palette-4-base u-btn-2">&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 447.243 447.243" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M420.361,192.229c-1.83-0.297-3.682-0.434-5.535-0.41H99.305l6.88-3.2c6.725-3.183,12.843-7.515,18.08-12.8l88.48-88.48    c11.653-11.124,13.611-29.019,4.64-42.4c-10.441-14.259-30.464-17.355-44.724-6.914c-1.152,0.844-2.247,1.764-3.276,2.754    l-160,160C-3.119,213.269-3.13,233.53,9.36,246.034c0.008,0.008,0.017,0.017,0.025,0.025l160,160    c12.514,12.479,32.775,12.451,45.255-0.063c0.982-0.985,1.899-2.033,2.745-3.137c8.971-13.381,7.013-31.276-4.64-42.4    l-88.32-88.64c-4.695-4.7-10.093-8.641-16-11.68l-9.6-4.32h314.24c16.347,0.607,30.689-10.812,33.76-26.88    C449.654,211.494,437.806,195.059,420.361,192.229z"></path>
</g>
</g></svg><img></span> Previous Unit
            </a>
          </div>
        </div>
      </div>
    </section>
    
    
    <footer class="u-align-center u-clearfix u-footer u-grey-80" id="sec-0012"><div class="u-clearfix u-sheet u-sheet-1"></div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
           
      <p class="u-text">This webpage was produced by Brian Poon as the Final Porfolio for class AS.410.610.82, &nbsp; <br>Bioinformatics: Tools for Genome Analysis</i>, at Johns Hopkins University.&nbsp;
      </p>
      
    </section>
  </body>
</html>