<!DOCTYPE html>
<html style="font-size: 16px;">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="description" content="">
    <meta name="page_type" content="np-template-header-footer-from-plugin">
    <title>Prokaryotic Genes</title>
    <link rel="stylesheet" href="nicepage.css" media="screen">
<link rel="stylesheet" href="Prokaryotic-Genes.css" media="screen">
    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="nicepage.js" defer=""></script>
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    <link id="u-page-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    
    
    
    
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "",
		"url": "index.html",
		"logo": "images/0-Logo_from_FreeLogoDesign.png"
}</script>
    <meta property="og:title" content="Prokaryotic Genes">
    <meta property="og:type" content="website">
    <meta name="theme-color" content="#478ac9">
    <link rel="canonical" href="index.html">
    <meta property="og:url" content="index.html">
  </head>
  <body class="u-body"><header class="u-clearfix u-header u-header" id="sec-9ffa"><div class="u-clearfix u-sheet u-sheet-1">
        <a href="Home.html" class="u-image u-logo u-image-1" data-image-width="183" data-image-height="83">
          <img src="images/0-Logo_from_FreeLogoDesign.png" class="u-logo-image u-logo-image-1" data-image-width="183">
        </a>
        <nav class="u-menu u-menu-dropdown u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><symbol id="menu-hamburger" viewBox="0 0 16 16" style="width: 16px; height: 16px;"><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</symbol>
</defs></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header> 
    <section class="u-clearfix u-valign-middle u-section-1" src="" id="sec-3f4f">
      <div class="u-expanded-height-lg u-expanded-height-md u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-ffba" style=""></use></svg>
        <svg class="u-svg-content" id="svg-ffba"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-clearfix u-layout-wrap u-layout-wrap-1">
        <div class="u-layout">
          <div class="u-layout-row">
            <div class="u-align-center u-container-style u-layout-cell u-left-cell u-size-25 u-layout-cell-1">
              <div class="u-container-layout u-container-layout-1">
                <div alt="" class="u-image u-image-circle u-image-1" data-image-width="1920" data-image-height="1280"></div>
              </div>
            </div>
            <div class="u-container-style u-layout-cell u-right-cell u-shape-rectangle u-size-35 u-layout-cell-2">
              <div class="u-container-layout u-container-layout-2">
                <h1 class="u-align-left u-custom-font u-text u-text-font u-title u-text-1">Prokaryotic Genes<br>And<br>Prediction&nbsp;<br>
                </h1>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-2" id="carousel_889b">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Prokaryotic Genes</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Prokaryotic mRNA is frequently polycistronic,
where one mRNA transcript results in multiple CDSs adjacent to each other. These
groups of genes are known as <i>operons</i> that has a single promoter sequence
regulating the transcription of several genes. By regulating the transcription
of the operon, an RNA polymerase can bind to the promoter and transcribe the
entire operon with all the genes on it. There are usually only one or two
nucleotides between the stop codon of one gene and start codon of the next gene
in a polycistronic mRNA transcript. Prokaryotic promoters will have RNA
polymerase bind to regions near the TSS. These regions are at -10 and -35 (relative
to the position of the TSS), and are likely to be binding sites for sigma
factor. An alpha-subunit on the RNA polymerase binds to AT-rich region near -40
to -60 called the <i>UP element</i>, which are usually found in strong
promoters.&nbsp;<br>&nbsp;&nbsp;<br>Sigma factors
are a subunit of RNA polymerase that binds to the promoter. Bacteria mostly use
sigma-70 under normal conditions, but during times of heat stress another sigma
factor, sigma-32, replaces sigma-70 in RNA polymerase. This causes the RNA
polymerase to bind to promoters specific for the genes of heat shock proteins
to increase their expression. Also, there are many more sigma factors, such as
sigma-28 (Chlamydia) and sigma-E (E. coli). Finding promoters will require the
use of training sets from promoters that are known known to bind to a specific
sigma factor. The use of intergenic regions of related species to find
promoters assumes that base substitutions are limited inside the promoter
because it must bind to the sigma factor. Luckily, prokaryotic promoters have
consensus sequences and can be found by statistical methods. Another methodic
approach is based on weaker stability of DNA double helix at promoters, which
is a normal part of promoter melting during transcription.&nbsp; &nbsp; &nbsp;&nbsp;<br>
                    <br>How are
prokaryotic genomes classified? Prokaryotic genomes can vary drastically
between species. Genome sizes can range from 0.5 megabases (Mb) to 10 Mb, with either
a circular or linear geometry. The prokaryotes themselves can have disease
significances and varying lifestyles that include being extracellular,
extremophilic, symbiotic, etc. Prokaryotic organisms can vary significantly in
GC content, which is expressed as a percentage with the formula (G+C)/total
bases, which may be attributed to the tendency for some organisms to prefer certain
codons over others. Although GC content are usually higher in genes, regions
that are atypical in GC content for the organism can be due to horizontal gene
transfer (HGT) that allows for interspecies transfer of genetic material. By comparison,
GC content remains the same during vertical gene transfer when genes are
inherited from parental cells. One commonality between all prokaryotes is the
presence of 16S rRNA subunit. This commonality allows the use of 16S rRNA to be
used for phylogenetic purposes in determining relationships among prokaryotic
species. However, it is important to note that this method is not fool-proof and
has challenges that will not be discussed here.&nbsp; &nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-3" id="carousel_f1cf">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Prokaryotic Gene Prediction</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Open reading frame (ORF) are the ways that a
sequence can be read. The ORF can vary between six locations from a sequence
start site, due to codons being composed of a triplet. The six locations are
stepwise patterns around a nucleotide position, moving to the next nucleotide position
(with the + or – indicating upstream or downstream). The predicted amino acid
sequences from one ORF will result in the sequence being read with a specific
nucleotide sequence as the start site for reading sequences. However, the start
site does not indicate the starting methionine, as it can start in the UTR. ORF
finding is used to find coding regions on prokaryotic genomic DNA. There are
(usually) no introns, so the longest open reading frame are the CDSs. The problem
is with ORF finding is determining the start codon when there are multiple
possibilities in the reading frame. Each AUG triplet can be either a start
codon, methionine, or from a non-coding region upstream of the start codon. Gene
prediction programs help with finding the most likely CDS regions from ORF
finding results. The accuracy of a gene prediction can be enhanced by using comparative
genomics, such as performing <i>BLASTx</i> to a protein database. However, one
disadvantage is that gene predictors cannot predict exact RNA sequences because
it can only distinguish between coding and noncoding DNA. This would not allow
it to determine transcription start site (TSS) or transcription termination
sites. This challenge can be resolved with RNA-seq experiments that provide
information on transcription start/stop sites in additional to gene expression
data). As such, RNA-seq experiments are often complimentary to gene prediction to
find the TSS that gene prediction cannot. Vice versa, gene prediction provides
the CDS regions that RNA-seq is not as capable of finding. Most modern
annotation methods will use gene predictors and transcription expression data in
order to annotate prokaryotic genes.&nbsp;&nbsp;
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-valign-middle u-section-4" id="carousel_a829">
      <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
      <div class="u-clearfix u-gutter-30 u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-row">
            <div class="u-size-60">
              <div class="u-layout-col">
                <div class="u-align-justify u-container-style u-layout-cell u-right-cell u-size-60 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <p class="u-text u-text-1">Some programs for
prokaryotic gene prediction include: <a href="https://www.ncbi.nlm.nih.gov/orffinder/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">ORF Finder</a>, <a href="http://www.softberry.com/berry.phtml?topic=fgenesb&amp;group=programs&amp;subgroup=gfindb" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">FGENESB</a>,
GLIMMER, <a href="http://exon.gatech.edu/GeneMark/gmhmmp.cgi" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">GeneMark</a>, and <a href="http://www.cbs.dtu.dk/services/EasyGene/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">EasyGene 1.2b</a>. ORF
Finder, as the name suggests, finds all available genes from each ORF read. The
“true” genes are usually the longest ORFs with non-overlapping ORFs. ORF Finder
will show the translated amino acid sequence when the ORF is selected.&nbsp;<br>
                    </p>
                    <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-2"></div>
                    <div class="u-shape u-shape-svg u-text-palette-1-base u-shape-3">
                      <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 130" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-5496"></use></svg>
                      <svg class="u-svg-content" id="svg-5496" style="enable-background:new 0 0 160 130;"><path d="M148.3,0H11.7C5.3,0,0,5.2,0,11.5v79.8c0,6.4,5.3,11.5,11.7,11.5h56L67.6,130l24.2-27.2h56.5c6.5,0,11.7-5.2,11.7-11.5V11.5
	C160,5.2,154.7,0,148.3,0z"></path></svg>
                    </div>
                    <h6 class="u-text u-text-white u-text-2">
                      <span style="font-size: 1.5rem;"><b>&nbsp;Practice Time!&nbsp;<br></b>
                      </span>
                      <br>Go
to the link for ORF Finder and copy the sequence from <a href="external/Bacillus%20subtilis%20genomic.txt" class="u-active-none u-border-2 u-border-active-palette-2-dark-1 u-border-hover-palette-2-base u-border-white u-btn u-button-link u-button-style u-hover-none u-none u-text-body-alt-color u-text-hover-palette-2-base u-btn-5">this</a> file that includes a FASTA format sequence for <i>Bacillus subtilis </i>genomic
DNA. Your result should look like below.&nbsp;&nbsp;
                    </h6>
                    <img src="images/4-ProkaryoticGenesAndAnnotation-OrfFinder_NCBI.png" alt="" class="u-image u-image-default u-image-1" data-image-width="1806" data-image-height="356">
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-5" id="carousel_e5fd">
      <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
      <div class="u-clearfix u-gutter-30 u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-row">
            <div class="u-size-60">
              <div class="u-layout-col">
                <div class="u-align-justify u-container-style u-layout-cell u-right-cell u-size-60 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <p class="u-text u-text-1">FGENSB is
a web-based online predictor program (also offered as a local download) based
on Markov chain models and can automatically annotate the predicted bacterial
genes through the GenBank database. This allows FGENESB to predict coding
regions, promoters, operons, translation sites, and termination sites.&nbsp;<br>
                    </p>
                    <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-2"></div>
                    <img src="images/4-ProkaryoticGenesAndAnnotation-FGENESB1.png" alt="" class="u-image u-image-default u-image-1" data-image-width="848" data-image-height="336">
                    <div class="u-shape u-shape-svg u-text-palette-1-base u-shape-3">
                      <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 130" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-d184"></use></svg>
                      <svg class="u-svg-content" id="svg-d184" style="enable-background:new 0 0 160 130;"><path d="M148.3,0H11.7C5.3,0,0,5.2,0,11.5v79.8c0,6.4,5.3,11.5,11.7,11.5h56L67.6,130l24.2-27.2h56.5c6.5,0,11.7-5.2,11.7-11.5V11.5
	C160,5.2,154.7,0,148.3,0z"></path></svg>
                    </div>
                    <h6 class="u-text u-text-white u-text-2">
                      <span style="font-size: 1.5rem;"><b>&nbsp;Practice Time!&nbsp;<br></b>
                      </span>
                      <br>&nbsp; Go to the link
for FGENESB and upload <a href="external/sheliprt.fasta" class="u-active-none u-border-2 u-border-active-palette-2-dark-1 u-border-hover-palette-2-base u-border-white u-btn u-button-link u-button-style u-hover-none u-none u-text-body-alt-color u-text-hover-palette-2-base u-btn-1">this</a> FASTA file. Select the training set (labelled as <i>Choose closest organism</i>)
as <i>BACTERIAL generic</i>. Your result should look like the image on the left.
Then, upload the sequence from <a href="external/lactococcus.txt" class="u-active-none u-border-2 u-border-active-palette-2-dark-1 u-border-hover-palette-2-base u-border-white u-btn u-button-link u-button-style u-hover-none u-none u-text-body-alt-color u-text-hover-palette-2-base u-btn-2">this</a> file and
select the training set as <i>Lactococcus lactis subsp lactis II1403</i>. Your
result should look like the image on the right.&nbsp; &nbsp;<br>
                    </h6>
                    <img src="images/4-ProkaryoticGenesAndAnnotation-FGENESB2.png" alt="" class="u-image u-image-default u-image-2" data-image-width="1266" data-image-height="730">
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-6" id="carousel_8089">
      <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
      <div class="u-clearfix u-gutter-30 u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-row">
            <div class="u-size-60">
              <div class="u-layout-col">
                <div class="u-align-justify u-container-style u-layout-cell u-right-cell u-size-60 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <p class="u-text u-text-1">GLIMMER can find long ORFs using genomic DNA that originates in a
FASTA format by utilizing training sets. Training sets are created from a
reference genome of the organism and comparing an unknown sequence from that
organism.&nbsp; If the DNA is unknown or a
whole reference genome is absent for the organism, then a reference genome from
a related organism can be used to create a training set. However, the prediction
may not be as strong depending on the relationship between different organisms.
For using training sets from related species, the related species must be of an
appropriate evolutionary distance. If it is too closely related, this can
result in false positives from not fully diverging. If it is too far in relationship,
this can result in only highly-conserved elements, such as with promoters, being
detected.&nbsp;&nbsp;<br>
                    </p>
                    <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-2"></div>
                    <img src="images/4-ProkaryoticGenesAndAnnotation-glimmer1.png" alt="" class="u-image u-image-default u-image-1" data-image-width="828" data-image-height="306">
                    <img src="images/4-ProkaryoticGenesAndAnnotation-glimmer2.png" alt="" class="u-image u-image-default u-image-2" data-image-width="920" data-image-height="314">
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-7" id="sec-7e4d">
      <div class="u-expanded-height u-flip-horizontal u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-3583"></use></svg>
        <svg class="u-svg-content" id="svg-3583"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-align-justify u-container-style u-group u-shape-rectangle u-white u-group-1">
        <div class="u-container-layout u-valign-bottom u-container-layout-1">
          <p class="u-text u-text-body-color u-text-1">Additional resources for prokaryotic and archaea gene annotation includes tools and databases such as: Seqinr, Pathosystems Resource Integration Center (<a href="http://www.patricbrc.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">PATRIC</a>),&nbsp;<a href="http://halo4.umbi.umd.edu/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">HaloWeb</a>, the&nbsp;<a href="http://archaea.ucsc.edu/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">UCSC Archaeal Genome Browser</a>,&nbsp;<a href="http://greengenes.lbl.gov/Download/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">Greengenes</a>,&nbsp;<a href="http://www.softberry.com/berry.phtml?topic=bprom&amp;group=programs&amp;subgroup=gfindb" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">BPROM</a>, and the&nbsp;<a href="http://www.fruitfly.org/seq_tools/promoter.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-6">Berkley Drosophila Genome Project</a>. Seqinr is a R-based package tool that allows for simple analysis of FASTA-formatted sequences, such as performing base count, sequence lengths, GC content, etc. PATRIC is a database of over 10,000 microbial genomes and allows genome browsing with cross-genome comparisons to find orthologs. HaloWeb is a database maintained by the University of Maryland’s Marine Biology Center that has information limited to 52 species of haloarchaea. The UCSC Archaeal Genome Browser is a separate genome browser from the widely-known UCSC Genome Browser that is specific for archaeal (and some bacterial) genomes. One benefit of this browser is that, like its widely-known counterpart, it is linkable to the Galaxy server. Lastly, Greengenes is a repository of 16S rRNA sequences with tools used for phylogenetic studies to compare (or identify in some cases) organisms. BPROM can be used to find potential sigma-70 promoters with a claimed accuracy of 80%. The Berkeley Drosophila Genome Project can predict promoters by utilizing a Neural Network Promoter Prediction algorithm that must be set for prokaryotes first. Unfortunately, as the field of bioinformatics grows and expands, some of these tools/databases may become outdated.&nbsp;&nbsp;
          </p>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-9" id="sec-d940">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-border-9 u-border-palette-1-light-2 u-container-style u-expanded-width u-group u-radius-50 u-shape-round u-group-1">
          <div class="u-container-layout u-container-layout-1">
            <h3 class="u-align-center u-text u-text-1">Prokaryotic Genes and Prediction</h3>
            <a href="Gene-Annotation-and-Structure.html" class="u-btn u-btn-round u-button-style u-hover-palette-4-base u-palette-1-base u-radius-2 u-btn-1">Next Unit&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 492.004 492.004" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M484.14,226.886L306.46,49.202c-5.072-5.072-11.832-7.856-19.04-7.856c-7.216,0-13.972,2.788-19.044,7.856l-16.132,16.136    c-5.068,5.064-7.86,11.828-7.86,19.04c0,7.208,2.792,14.2,7.86,19.264L355.9,207.526H26.58C11.732,207.526,0,219.15,0,234.002    v22.812c0,14.852,11.732,27.648,26.58,27.648h330.496L252.248,388.926c-5.068,5.072-7.86,11.652-7.86,18.864    c0,7.204,2.792,13.88,7.86,18.948l16.132,16.084c5.072,5.072,11.828,7.836,19.044,7.836c7.208,0,13.968-2.8,19.04-7.872    l177.68-177.68c5.084-5.088,7.88-11.88,7.86-19.1C492.02,238.762,489.228,231.966,484.14,226.886z"></path>
</g>
</g></svg><img></span>
            </a>
            <a href="Eukaryotic-Genes.html" class="u-btn u-btn-round u-button-style u-hover-palette-1-light-2 u-palette-1-base u-radius-2 u-text-hover-palette-4-base u-btn-2">&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 447.243 447.243" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M420.361,192.229c-1.83-0.297-3.682-0.434-5.535-0.41H99.305l6.88-3.2c6.725-3.183,12.843-7.515,18.08-12.8l88.48-88.48    c11.653-11.124,13.611-29.019,4.64-42.4c-10.441-14.259-30.464-17.355-44.724-6.914c-1.152,0.844-2.247,1.764-3.276,2.754    l-160,160C-3.119,213.269-3.13,233.53,9.36,246.034c0.008,0.008,0.017,0.017,0.025,0.025l160,160    c12.514,12.479,32.775,12.451,45.255-0.063c0.982-0.985,1.899-2.033,2.745-3.137c8.971-13.381,7.013-31.276-4.64-42.4    l-88.32-88.64c-4.695-4.7-10.093-8.641-16-11.68l-9.6-4.32h314.24c16.347,0.607,30.689-10.812,33.76-26.88    C449.654,211.494,437.806,195.059,420.361,192.229z"></path>
</g>
</g></svg><img></span> &nbsp;Previous Unit
            </a>
          </div>
        </div>
      </div>
    </section>
    
   <footer class="u-align-center u-clearfix u-footer u-grey-80" id="sec-0012"><div class="u-clearfix u-sheet u-sheet-1"></div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
           
      <p class="u-text">This webpage was produced by Brian Poon as the Final Porfolio for class AS.410.610.82, &nbsp; <br>Bioinformatics: Tools for Genome Analysis</i>, at Johns Hopkins University.&nbsp;
      </p>
      
    </section>
  </body>
</html>