<!DOCTYPE html>
<html style="font-size: 16px;">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="description" content="">
    <meta name="page_type" content="np-template-header-footer-from-plugin">
    <title>RNA-seq</title>
    <link rel="stylesheet" href="nicepage.css" media="screen">
<link rel="stylesheet" href="RNA-seq.css" media="screen">
    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="nicepage.js" defer=""></script>
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    
    
    
    
    
    
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "",
		"url": "index.html",
		"logo": "images/0-Logo_from_FreeLogoDesign.png"
}</script>
    <meta property="og:title" content="RNA-seq">
    <meta property="og:type" content="website">
    <meta name="theme-color" content="#478ac9">
    <link rel="canonical" href="index.html">
    <meta property="og:url" content="index.html">
  </head>
  <body class="u-body"><header class="u-clearfix u-header u-header" id="sec-9ffa"><div class="u-clearfix u-sheet u-sheet-1">
        <a href="Home.html" class="u-image u-logo u-image-1" data-image-width="183" data-image-height="83">
          <img src="images/0-Logo_from_FreeLogoDesign.png" class="u-logo-image u-logo-image-1" data-image-width="183">
        </a>
        <nav class="u-menu u-menu-dropdown u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><symbol id="menu-hamburger" viewBox="0 0 16 16" style="width: 16px; height: 16px;"><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</symbol>
</defs></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header> 
    <section class="u-clearfix u-valign-middle u-section-1" src="" id="sec-3f4f">
      <div class="u-expanded-height-lg u-expanded-height-md u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-ffba" style=""></use></svg>
        <svg class="u-svg-content" id="svg-ffba"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-clearfix u-layout-wrap u-layout-wrap-1">
        <div class="u-layout">
          <div class="u-layout-row">
            <div class="u-align-center u-container-style u-layout-cell u-left-cell u-size-25 u-layout-cell-1">
              <div class="u-container-layout u-container-layout-1">
                <div alt="" class="u-image u-image-circle u-image-1" data-image-width="1920" data-image-height="1280"></div>
              </div>
            </div>
            <div class="u-container-style u-layout-cell u-right-cell u-shape-rectangle u-size-35 u-layout-cell-2">
              <div class="u-container-layout u-container-layout-2">
                <h1 class="u-align-left u-custom-font u-text u-text-font u-title u-text-1">RNA sequencing (RNA-seq)<br>
                </h1>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-2" id="carousel_889b">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">RNA-seq</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">When compared
to the previous chapter on ChIP-seq, RNA-seq employs the same initial principle
of obtaining a target of interest and, more often than not, aligning the sequence
data to a reference genome. In RNA-seq, the target of interest is the <i>transcriptome</i> of the sample. The transcriptome encompasses all the cellular RNA transcripts,
such as mRNA, tRNA, ncRNA, etc. RNA-seq analyses reveals locations, and levels,
of known and novel transcripts by providing quantitative information on gene
expression across cell types and sample conditions. New transcripts expressed
under rare conditions are still being discovered. The uses for RNA-seq are enormous
in that it can define transcript start sites, reveal new ncRNAs, find alterative
splice products or sites, and- most importantly- quantitatively estimate
transcript levels. The latter is the most important in that this particular attribute
to RNA-seq, as well as improvements in technology that reduces the cost,
ultimately results in RNA-seq replacing traditional microarrays. Microarrays
were widely used, until recently, to measure the relative gene expression of
the cell due to the background noise generated, but fails to quantitatively
measure the gene expression. Comparatively, RNA-seq has the resolution of a
single base-pair with relatively low background noise. Microarrays are good for
general RNA expression levels, but still need very specific probes on the array
and the resolution is not adequate for detecting differential splicing
expression.<b></b>Financially, RNA-seq is more expensive with a cost of
$200-$300 per sample, but the benefit is that it produces quantitative data of
every cDNA that can be sequenced<b>. </b>Comparatively, expression microarrays can
only be used for approximately 20,000 known genes and splice variants, with a
cost of about $50-$200 per sample for statistically sound results.&nbsp;<b></b>
                    <br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-3" id="carousel_f1cf">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-valign-top u-container-layout-1">
                  <p class="u-align-justify u-text u-text-1">Contrary to the
name, RNA-seq is the term for massive parallel sequencing of cDNA- not the RNA.
This is because the RNA themselves are the templates for a cDNA library to
convert the RNA to cDNA, which is used by NGS platforms to sequence the reads. Following
similar principles, the sequenced reads are then aligned to a reference genome
to determine the location of the original RNA transcript. Unless the RNA
transcript is foreign, such as with RNA-based viruses, the transcript must have
been transcribed from a genomic location on the DNA. Watch the video below for
more in-depth details on RNA-seq and analysis.&nbsp; &nbsp;<br>
                  </p>
                  <div class="u-video u-video-1 wrap">
                    <div class="embed-responsive embed-responsive-1">
                      <iframe style="width: 100%;height: 100%;" class="embed-responsive-item frame" src="https://www.youtube.com/embed/hksQlJLwKqo" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-4" id="carousel_f819">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-valign-bottom u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Reference-Based RNA-seq</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">There are many
tools available in performing reference-based RNA-seq analysis. Most RNA-seq
experiments aim to understand the transcriptome of the sample. In which case, comparative
analysis of differential expression of the transcriptome between two samples is
the central theme. The most commonly used, and well-known, is the <i>
                      <a href="https://ccb.jhu.edu/software/tophat/index.shtml" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">TopHat</a></i>suite,
which includes a <i>Bowtie</i> alignment program as part of the tools. The RNA-seq
analysis pipeline can be summarized in five major steps, albeit certain steps
may be repetitive upon itself. Given that we have the sequenced reads of a
sample under two different conditions, the first step in differential RNA
expression analysis is to use the two reads as the input to <i>TopHat</i>,
which will map the reads using the <i>Bowtie </i>aligner with an exon-based
approach. The mapped reads of each condition will become the input into the <i>
                      <a href="http://cole-trapnell-lab.github.io/cufflinks/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">Cufflinks</a></i> tool, as
step two, which assembles the reads into transcripts using a probabilistic
model. A <i>general transfer format</i> (<a href="http://uswest.ensembl.org/info/website/upload/gff.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">GTF</a>) file that
contains information on gene location and names can be incorporated with <i>Cufflinks</i> to attach those features in the GTF file to the transcripts. The outputs from <i>Cufflinks</i> are two GTF files, with two tables showing gene expression or transcript
expression levels. The third step is to use <i>
                      <a href="http://cole-trapnell-lab.github.io/cufflinks/cuffmerge/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">Cuffmerge</a></i> to incorporate the two files of mapped transcripts into a final, single
transcriptome assembly. The output of <i>Cuffmerge</i> goes into <i>
                      <a href="http://cole-trapnell-lab.github.io/cufflinks/cuffdiff/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">Cuffdiff</a></i> in step four for statistical analysis of differential expression between the
two conditions. The mapped reads outputted from <i>TopHat</i> must be inputted
with <i>Cuffdiff</i> and the transcriptome assembly for comparison. The result
from <i>Cuffdiff</i> is the results of the analysis. Lastly, in step five, the
differential expression results from <i>Cuffdiff</i> goes into <i>
                      <a href="http://compbio.mit.edu/cummeRbund/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-6">CummeRbund</a></i> to be plotted for
visually examining the differential expression results. Please see the figure
below for the general pipeline of RNA-seq differential expression analysis. Although
this is not the third step, and is not shown in the figure, the <i>
                      <a href="http://cole-trapnell-lab.github.io/cufflinks/cuffcompare/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-7">Cuffcompare</a></i> tool can be used prior to <i>Cuffmerge</i> to compare the two GTF files from <i>Cufflinks</i>.
The goal of comparing may be to compare gene expression levels between
different cell types, different developmental stages, or even different disease
states. The <i>TopHat</i> suite of tools are available on Galaxy.&nbsp; &nbsp;<br>
                  </p>
                  <img src="images/12-RNASeq-TopHatPipeline_class.png" alt="" class="u-image u-image-default u-image-1" data-image-width="250" data-image-height="571">
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-5" id="carousel_5cbc">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-valign-top u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Differential Expression and <i>De novo </i>Assemblers
                  </h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Challenges can
occur when aligning cDNA, that is specific to aligning cDNA and do not occur
with genomic DNA sequencing. Most sequenced eukaryotic cDNA reads are from
spliced mRNA. Thus, a single read can span an exon-exon boundary with a large gap
in alignment from the intron that is missing. This is especially evident with
paired-end sequencing when paired reads could appear very far apart on the
reference genome. The major challenge lies in assembling the reads into a transcript.
This is a unique feature of RNA-seq alignment programs that specializes in overcoming
these challenges.&nbsp; &nbsp;<br>
                    <br>&nbsp; As we approach
the more modern-day methods and technology, we must keep in mind that new technologies,
approaches, and methods for analyzing RNA-seq data are continuously emerging. One
new tool is the <i>
                      <a href="http://www.ncbi.nlm.nih.gov/pubmed/22266656" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">MATS</a></i> package, which is a statistical analysis method for the hypothesis testing of
alternative splice patterns detected by RNA-seq. Another tool is <i>
                      <a href="http://www.ncbi.nlm.nih.gov/pubmed/23155066" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">DiffSplice</a></i>, which
recently found new alternative splicing events that were not reported to the RefSeq
or UCSC Genes databases. <i>Diffsplice</i> compares the transcriptomes assembled
by multiple RNA-seq datasets to find novel splice transcripts. <i>
                      <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0550-8" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">DESeq2</a></i> is another tool that is popular for use in differential expression analysis
that is available through Bioconductor. <i>
                      <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btp616" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">EdgeR</a></i>is another tool that can be found in Bioconductor. <i>
                      <a href="https://ccb.jhu.edu/software/stringtie/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">StringTie</a></i> is a tool
developed at Johns Hopkins University that performs transcript assembly and
expression analysis in a single step, thereby potentially replacing <i>Cufflinks</i>.
Regardless of the tool, they all have one commonality: they rely on the RNA-seq
reads to be aligned to a reference genome.&nbsp;<b></b>
                    <br>
                    <br>&nbsp; What is the
reference genome is not available? We have covered situations in NGS where <i>de
novo</i> alignment had to be performed for newly sequenced genomes. RNA-seq alignment
programs also have <i>de novo</i> assemblers that can be used before transcript
assembly and quantification. Similar to <i>de novo </i>NGS aligners, RNA-seq <i>de
novo </i>assemblers will align the RNA-seq reads to each other based on
overlapping segments to determine what the full RNA molecule must have been. One
popular <i>de novo</i> assembler for RNA-seq is <i>
                      <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3571712/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-6">Trinity</a></i>.
However, these tools tend to be newer than reference-based assemblers and are actively
being developed.&nbsp; &nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-6" id="carousel_69b8">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Single-cell RNA-seq</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">In addition to the
general RNA-seq, a new and growing approach is <i>single-cell RNA-seq</i> (scRNA-seq). As the name implies, the goal of this method is to measure the
transcriptome of a single cell using <a href="https://www.pacb.com/smrt-science/smrt-sequencing/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">SMRT-seq library</a> preparation method. For comparison purposes, a single human cell has at least
20,000 genes and by doing RNA-seq on a single cell, the result can show the
expression of at least 20,000 genes. Advancements in NGS technology allows
RNA-seq to be performed once, without the number of reads being an issue. The
first step to scRNA-seq is to insert a specific barcode sequence, which is usually
12 base-pairs long, in order to determine the origination of the sequence read.
A common method of incorporating the barcode sequence into a single cell is
through <i>
                      <a href="https://www.illumina.com/science/sequencing-method-explorer/kits-and-arrays/drop-seq.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">Drop-seq</a></i>,
which encapsulates a cell, and a bead with the barcode sequence attached, in an
aqueous droplet that is formed in an oily environment within a microfluidic
channel. The cellular membrane is ruptured for a few seconds with detergent and
the mRNAs in the cell are captured with the polyA part of the arms in the bead.
Lastly, the cDNA is generated using the SMRT-seq protocol mentioned prior.&nbsp;<br>
                    <br>In addition to
containing the same cell-specific barcode sequence in the arms of the bead, it
also contains a <i>unique molecular modifier</i> (UMI) that is different
amongst each <u>arm</u> of the bead. The length of the UMI has since been
elongated to 16 base-pairs as of 2021. After Drop-seq is performed, paired-end
sequencing performed for scRNA-seq using Illumina. The first read is for cell
barcode and UMI, while second read is for cDNA. After the reads are sequenced,
they are sorted based on the barcode sequence in the first read before the second
sequence read is mapped. The number of UMI mapped to each gene is calculated
and saved as a count matrix, with each columns representing a single cell and
rows representing the genes. The count matrix will contain many “0” values,
which means no expression. In order to save disk space, the <a href="https://hbctraining.github.io/scRNA-seq/lessons/readMM_loadData.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">MTX</a> file format is used to only denote non-zero values from the count matrix. The lines
in a MTX file format are in X, Y, Z format, with X and Y denoting the matrix coordinate
and Z denoting the non-zero value. Lastly, the MTX file has two corresponding TSV
files that go with it: one for genes and the other for the barcode sequences.
This way, the genes are mapped to the row of the matrix and the barcode
sequence for the cell it originates from.&nbsp;&nbsp;<br>
                  </p>
                  <img src="images/12-RNASeq-scRNAseq_class.png" alt="" class="u-image u-image-default u-image-1" data-image-width="600" data-image-height="450">
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-7" id="carousel_4cf3">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-valign-top u-container-layout-1">
                  <p class="u-align-justify u-text u-text-1">After obtaining
the UMI count matrix, quality control performed to remove noise and multilets. Noise
is where no cell gets into a droplet and multilets occur when two or more cells
get into droplet. To identify noise, a kneeplot, which is the cumulative plot
of the number of UMIs per cell in decreasing order, is used to determine a
cutoff value for noise based on when the graph shifts exponentially. This makes
the cumulative version of the UMI/cell plot. 
Afterwards,
normalize the number of UMI by dividing total number of UMIs to the number of
cells. The normalized values will be very small and will need to be scaled up. Scale
the result up by multiplying 10,000, and take either natural log or log2, which
is dependent upon the software. Next, perform dimensionality reduction or
clustering analysis with <i>Principal Component Analysis</i> (PCA) and, either
t-SNE or UMAP. The <a href="https://satijalab.org/seurat/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Seurat</a> program
can perform PCA to clusters and visualize by t-SNE or UMAP. This can be used to
identify the marker genes per cluster in order to assign cell type. Lastly,
perform differential expression analysis for cell-type. &nbsp;&nbsp;&nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-8" id="sec-7e4d">
      <div class="u-expanded-height u-flip-horizontal u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-3583"></use></svg>
        <svg class="u-svg-content" id="svg-3583"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-align-justify u-container-style u-group u-shape-rectangle u-white u-group-1">
        <div class="u-container-layout u-valign-top u-container-layout-1">
          <p class="u-text u-text-body-color u-text-1">A potential
problem with scRNA-seq is that when library preparation is done for single cell,
the reverse transcriptase often gets stalled in the middle. This can cause failures
in generating a full-length cDNA. One solution is to use MMLV (Moloney Murine
Leukemia Virus) reverse transcriptase, because it adds the CCC trinucleotide when
it reaches 5’ end of mRNA template. This would require the use of a PCR handle with
the oligonucleotide GGG added, because it is complimentary to the CCC
oligonucleotide and only strands with PCR handles inserted can be amplified to generate
full-length cDNA. Watch the lecture below for information on alignment programs for both ChIP-seq and RNA-seq.</p>
        </div>
      </div>
      <div class="u-video u-video-1 wrap">
        <div class="embed-responsive embed-responsive-1">
          <iframe style="width: 100%;height: 100%;" class="embed-responsive-item frame" src="https://www.youtube.com/embed/2qGiw4MRK3c" title="YouTube video player" frameborder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-9" id="sec-d940">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-border-9 u-border-palette-1-light-2 u-container-style u-expanded-width u-group u-radius-50 u-shape-round u-group-1">
          <div class="u-container-layout u-container-layout-1">
            <h3 class="u-align-center u-text u-text-1">RNA-seq</h3>
            <a href="ForwardOntoDawn.html" class="u-btn u-btn-round u-button-style u-hover-palette-4-base u-palette-1-base u-radius-2 u-btn-1">Next Unit&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 492.004 492.004" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M484.14,226.886L306.46,49.202c-5.072-5.072-11.832-7.856-19.04-7.856c-7.216,0-13.972,2.788-19.044,7.856l-16.132,16.136    c-5.068,5.064-7.86,11.828-7.86,19.04c0,7.208,2.792,14.2,7.86,19.264L355.9,207.526H26.58C11.732,207.526,0,219.15,0,234.002    v22.812c0,14.852,11.732,27.648,26.58,27.648h330.496L252.248,388.926c-5.068,5.072-7.86,11.652-7.86,18.864    c0,7.204,2.792,13.88,7.86,18.948l16.132,16.084c5.072,5.072,11.828,7.836,19.044,7.836c7.208,0,13.968-2.8,19.04-7.872    l177.68-177.68c5.084-5.088,7.88-11.88,7.86-19.1C492.02,238.762,489.228,231.966,484.14,226.886z"></path>
</g>
</g></svg><img></span>
            </a>
            <a href="ChIP-seq.html" class="u-btn u-btn-round u-button-style u-hover-palette-1-light-2 u-palette-1-base u-radius-2 u-text-hover-palette-4-base u-btn-2">&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 447.243 447.243" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M420.361,192.229c-1.83-0.297-3.682-0.434-5.535-0.41H99.305l6.88-3.2c6.725-3.183,12.843-7.515,18.08-12.8l88.48-88.48    c11.653-11.124,13.611-29.019,4.64-42.4c-10.441-14.259-30.464-17.355-44.724-6.914c-1.152,0.844-2.247,1.764-3.276,2.754    l-160,160C-3.119,213.269-3.13,233.53,9.36,246.034c0.008,0.008,0.017,0.017,0.025,0.025l160,160    c12.514,12.479,32.775,12.451,45.255-0.063c0.982-0.985,1.899-2.033,2.745-3.137c8.971-13.381,7.013-31.276-4.64-42.4    l-88.32-88.64c-4.695-4.7-10.093-8.641-16-11.68l-9.6-4.32h314.24c16.347,0.607,30.689-10.812,33.76-26.88    C449.654,211.494,437.806,195.059,420.361,192.229z"></path>
</g>
</g></svg><img></span> &nbsp;Previous Unit
            </a>
          </div>
        </div>
      </div>
    </section>
    
    
     <footer class="u-align-center u-clearfix u-footer u-grey-80" id="sec-0012"><div class="u-clearfix u-sheet u-sheet-1"></div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
           
      <p class="u-text">This webpage was produced by Brian Poon as the Final Porfolio for class AS.410.610.82, &nbsp; <br>Bioinformatics: Tools for Genome Analysis</i>, at Johns Hopkins University.&nbsp;
      </p>
      
    </section>
  </body>
</html>