<!DOCTYPE html>
<html style="font-size: 16px;">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="description" content="">
    <meta name="page_type" content="np-template-header-footer-from-plugin">
    <title>RNA and Ultraconserved Regions</title>
    <link rel="stylesheet" href="nicepage.css" media="screen">
<link rel="stylesheet" href="RNA-and-Ultraconserved-Regions.css" media="screen">
    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="nicepage.js" defer=""></script>
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    
    
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "",
		"url": "index.html",
		"logo": "images/0-Logo_from_FreeLogoDesign.png"
}</script>
    <meta property="og:title" content="RNA and Ultraconserved Regions">
    <meta property="og:type" content="website">
    <meta name="theme-color" content="#478ac9">
    <link rel="canonical" href="index.html">
    <meta property="og:url" content="index.html">
  </head>
  <body class="u-body"><header class="u-clearfix u-header u-header" id="sec-9ffa"><div class="u-clearfix u-sheet u-sheet-1">
        <a href="Home.html" class="u-image u-logo u-image-1" data-image-width="183" data-image-height="83">
          <img src="images/0-Logo_from_FreeLogoDesign.png" class="u-logo-image u-logo-image-1" data-image-width="183">
        </a>
        <nav class="u-menu u-menu-dropdown u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><symbol id="menu-hamburger" viewBox="0 0 16 16" style="width: 16px; height: 16px;"><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</symbol>
</defs></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header> 
    <section class="u-clearfix u-valign-middle u-section-1" src="" id="sec-3f4f">
      <div class="u-expanded-height-lg u-expanded-height-md u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-ffba" style=""></use></svg>
        <svg class="u-svg-content" id="svg-ffba"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-clearfix u-layout-wrap u-layout-wrap-1">
        <div class="u-layout">
          <div class="u-layout-row">
            <div class="u-align-center u-container-style u-layout-cell u-left-cell u-size-25 u-layout-cell-1">
              <div class="u-container-layout u-container-layout-1">
                <div alt="" class="u-image u-image-circle u-image-1" data-image-width="1920" data-image-height="1280"></div>
              </div>
            </div>
            <div class="u-container-style u-layout-cell u-right-cell u-shape-rectangle u-size-35 u-layout-cell-2">
              <div class="u-container-layout u-container-layout-2">
                <h1 class="u-align-left u-custom-font u-text u-text-font u-title u-text-1">RNA And Ultraconserved Regions</h1>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-2" id="carousel_889b">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">RNA</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">RNA are more
than the transcripts that are usually considered as the “middle-man” between
genes and proteins. One of the most critical roles they play was not strongly
known, nor appreciated, until early in the 21st century and the role
is noncoding RNA (ncRNA). Although there are still much to learn about ncRNA,
we do have much appreciation for their ability in fine-tune regulation of gene
expression. However, the term ncRNA is defined as RNA molecule transcripts that
do not translate into protein and broadly encompasses multiple classes of
noncoding RNA: transfer RNA (tRNA), ribosomal RNA (rRNA), long noncoding RNAs (lncRNAs),
small nucleolar RNA (snoRNA), small nuclear RNA (snRNA), microRNA (miRNA),
short interfering RNA (siRNA), and guide RNA (gRNA). Even today, more classes
of ncRNAs are being discovered. However, omitted from the list here are messenger
RNA (mRNA), which are translated into protein. Compared to previously believed,
studies show that lncRNA may be more ubiquitous with possible roles in
transcriptional regulation, epigenetics, and brain function. &nbsp;&nbsp;&nbsp;<br>
                    <br>A new term that
may be used synonymously with ncRNA is the term functional RNA (fRNA). However,
fRNA is broader, encompassing both ncRNA and functional secondary structures in
mRNAs. Some examples of fRNA that are not ncRNAs include riboswitches and the selenocysteine
insertion sequence (SECIS) element. One program focused on RNA is PPfold, which
can accept multiple alignments of RNAs and uses phylo-SCFG methods focused around
entropy to predict RNA folding, as described by <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S2-S22" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Sükösd
et al. (2013)</a>.&nbsp;<b></b>
                    <br>
                    <br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-3" id="carousel_f1cf">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">RNA Prediction and Resources</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Some tools for predicting
miRNA targets include: miRanda, <a href="http://pictar.mdc-berlin.de/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">PicTar</a>,
and <a href="http://carolina.imis.athena-innovation.gr/diana_tools/web/index.php?r=tarbasev8%2Findex" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">DIANA-TarBase
v.8</a>. The algorithm miRanda finds miRNA and miRNA targets on an mRNA by
looking for target sites on the 3’ UTR using comparative genomics. The PicTar algorithm
was described in a <i>
                      <a href="http://www.nature.com/ng/journal/v37/n5/abs/ng1536.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">Nature Genetics</a></i> article and has pre-calculated predictions for miRNA. The DIANA-TarBase v.8
algorithm predicts miRNA-to-mRNA targeting, with the input for either miRNA or
mRNA needed, and includes links to external sources.&nbsp;<br>
                    <br>Some databases
that contain information for ncRNA include <a href="http://mirdb.org/miRDB/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">miRDB</a>, <a href="http://www.mirbase.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">miRBase</a>, <a href="http://www.targetscan.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-6">TargetScan</a>, <a href="http://rhesus.amu.edu.pl/mirnest/copy/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-7">miRNEST 2.0</a>, <a href="http://compbio.uthsc.edu/miRSNP/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-8">PolymiRTS</a>, <a href="http://www.noncode.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-9">NONCODE</a>, and <a href="http://rfam.xfam.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-10">Rfam</a>.
The TargetScan database contains precalculated miRNA predictions that are
separated by species and gene symbol. The miRNEST 2.0 database contains data
from deep sequencing libraries and predictions of animal and plant miRNA. PolymiRTS
contain data on SNPs in miRNA target sites for both known and predicted sites, in
which case a SNP located at an important base-pair will interfere with
miRNA-based gene silencing. The miRDB is a database that allows searching of a
given mRNA to return miRNA actors, and vice versa. The miRBase is a database
for known miRNA sequences, with detailed records, annotation, and data from
deep sequencing reads. The NONCODE database is a manually curated database with
records of lncRNA transcripts from lncrnadb for over 500,000 sequences across
16 species. Rfam is a database hosted by the European Bioinformatics Institute
(EBI) that is a ncRNA-based version of Pfam. Rfam includes information on
structure and function for ncRNA categorized into <i>Families</i>, <i>Clans</i>, <i>Motifs</i>, and structure.&nbsp;&nbsp;
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-4" id="sec-be2c">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-valign-bottom u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1"> Ultraconserved Elements</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">The term <i>Ultraconserved
Element</i> (UCE) was first described in 2004 by&nbsp;<a href="https://www.ncbi.nlm.nih.gov/pubmed/15131266" target="_blank" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Bejerano&nbsp;et al.</a> as regions
of over 200 base-pairs that have 100% identity with no gaps in human, mouse,
and rat- the identity and length of which cannot be statistically attributed to
chance. While many of these UCEs are not found in coding regions, the UCEs
found in non-coding regions were referred to as “gene deserts”. Incidentally,
the 100% identity requirement for UCEs ensure that SNPs are extremely rare,
even though the variant may not be lethal. Another interest to note about UCEs
is that they are rarely detected outside of chordates, which show how conserved
they are in evolutionary terms. UCEs may have functions attributed to them,
such as <i>uc322</i>, which was found to be an intron of the <i>SOX6</i> gene
that appears to be an enhancer in melanocytes. Many discovered UCEs were linked
to tumorigenesis were attributed to a new class of ncRNAs called transcribed
ultraconserved regions (T-UCR), indicating the delicate responsibilities that
these gene regions have in the organism.&nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-5" id="carousel_00be">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-valign-bottom u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1"> Ultraconserved Elements and Conserved Noncoding Elements </h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">The terms UCEs
and T-UCRs are very restricting in their definitions. In which case, there are
two other terms for a conserved, but less stringent region. The first is called <i>long conserved noncoding sequences</i> (LCNS) and are defined as sequences
longer than 500 base-pairs with more than 95% identity. The second is <i>highly
conserved noncoding sequences</i> (CNEs) that is more of a modern focus with
less limitations, as described by <a href="http://dx.doi.org/10.1098/rstb.2013.0021" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Harmston et al.</a> in 2013, by
defining CNEs based on the pair of species that are compared against each
other.&nbsp;<br>
                    <br>Tools for researching CNEs include the databases:&nbsp;<a href="http://ancora.genereg.net/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">ANCORA</a>,&nbsp;<a href="http://bioinformatics.bc.edu/chuanglab/cneViewer/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">cneViewer</a>,&nbsp;<a href="http://ccg.vital-it.ch/UCNEbase/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">UCNEbase</a>, and&nbsp;<a href="https://cega.ezlab.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">CEGA</a>. The ANCORA database has a genome browser for 11 species to compare with each other with the option of downloading the data or exporting the tracks to the UCSC Genome Browser or Ensembl. In addition, cneViewer is a genome viewer that displays conserved regions between only humans and zebrafish. Comparatively, UCNEbase contains records of over 4,000&nbsp;<i>ultraconserved non-coding elements</i>&nbsp;(UCNEs), a new term used by&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531063/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-6">Dimitrieva and Bucher</a>&nbsp;to describe most conserved non-coding elements, across 18 species and is more advanced/comprehensive when compared to other current databases. CEGA is a newly established database with data on similarly-based regions using another term:&nbsp;<i>conserved non-coding sequences</i>&nbsp;(CNCs). At the end of this paragraph, one must think that there is numerous terminology all focused around the same conceptual idea, with specific parameters given that altercate what encompasses one and not the other. What is the order? Will it change? It’s not surprising those questions comes up in your mind. This topic, as well as the database and resources that accompany it, are new, relative to the date of this being written. Compared to other subjects, the research on these conserved (whether ultra or not) are still minimal and the data is young. These terms and resources will change by the time you are reading this. However, this tool may not be outdated- it’s just the start to helping you understand the history behind this topic.&nbsp; &nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-9" id="sec-d940">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-border-9 u-border-palette-1-light-2 u-container-style u-expanded-width u-group u-radius-50 u-shape-round u-group-1">
          <div class="u-container-layout u-container-layout-1">
            <h3 class="u-align-center u-text u-text-1">RNA and Ultraconserved Regions</h3>
            <a href="Variants.html" class="u-btn u-btn-round u-button-style u-hover-palette-4-base u-palette-1-base u-radius-2 u-btn-1">Next Unit&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 492.004 492.004" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M484.14,226.886L306.46,49.202c-5.072-5.072-11.832-7.856-19.04-7.856c-7.216,0-13.972,2.788-19.044,7.856l-16.132,16.136    c-5.068,5.064-7.86,11.828-7.86,19.04c0,7.208,2.792,14.2,7.86,19.264L355.9,207.526H26.58C11.732,207.526,0,219.15,0,234.002    v22.812c0,14.852,11.732,27.648,26.58,27.648h330.496L252.248,388.926c-5.068,5.072-7.86,11.652-7.86,18.864    c0,7.204,2.792,13.88,7.86,18.948l16.132,16.084c5.072,5.072,11.828,7.836,19.044,7.836c7.208,0,13.968-2.8,19.04-7.872    l177.68-177.68c5.084-5.088,7.88-11.88,7.86-19.1C492.02,238.762,489.228,231.966,484.14,226.886z"></path>
</g>
</g></svg><img></span>
            </a>
            <a href="Eukaryotic-Genes.html" class="u-btn u-btn-round u-button-style u-hover-palette-1-light-2 u-palette-1-base u-radius-2 u-text-hover-palette-4-base u-btn-2">&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 447.243 447.243" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M420.361,192.229c-1.83-0.297-3.682-0.434-5.535-0.41H99.305l6.88-3.2c6.725-3.183,12.843-7.515,18.08-12.8l88.48-88.48    c11.653-11.124,13.611-29.019,4.64-42.4c-10.441-14.259-30.464-17.355-44.724-6.914c-1.152,0.844-2.247,1.764-3.276,2.754    l-160,160C-3.119,213.269-3.13,233.53,9.36,246.034c0.008,0.008,0.017,0.017,0.025,0.025l160,160    c12.514,12.479,32.775,12.451,45.255-0.063c0.982-0.985,1.899-2.033,2.745-3.137c8.971-13.381,7.013-31.276-4.64-42.4    l-88.32-88.64c-4.695-4.7-10.093-8.641-16-11.68l-9.6-4.32h314.24c16.347,0.607,30.689-10.812,33.76-26.88    C449.654,211.494,437.806,195.059,420.361,192.229z"></path>
</g>
</g></svg><img></span> &nbsp;Previous Unit
            </a>
          </div>
        </div>
      </div>
    </section>
    
    <footer class="u-align-center u-clearfix u-footer u-grey-80" id="sec-0012"><div class="u-clearfix u-sheet u-sheet-1"></div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
           
      <p class="u-text">This webpage was produced by Brian Poon as the Final Porfolio for class AS.410.610.82, &nbsp; <br>Bioinformatics: Tools for Genome Analysis</i>, at Johns Hopkins University.&nbsp;
      </p>
      
    </section>
  </body>
</html>