<!DOCTYPE html>
<html style="font-size: 16px;">
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta charset="utf-8">
    <meta name="description" content="">
    <meta name="page_type" content="np-template-header-footer-from-plugin">
    <title>Variants</title>
    <link rel="stylesheet" href="nicepage.css" media="screen">
<link rel="stylesheet" href="Variants.css" media="screen">
    <script class="u-script" type="text/javascript" src="jquery.js" defer=""></script>
    <script class="u-script" type="text/javascript" src="nicepage.js" defer=""></script>
    <link id="u-theme-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,400,400i,500,500i,700,700i,900,900i|Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    <link id="u-page-google-font" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i">
    
    
    
    
    
    
    
    
    <script type="application/ld+json">{
		"@context": "http://schema.org",
		"@type": "Organization",
		"name": "",
		"url": "index.html",
		"logo": "images/0-Logo_from_FreeLogoDesign.png"
}</script>
    <meta property="og:title" content="Variants">
    <meta property="og:type" content="website">
    <meta name="theme-color" content="#478ac9">
    <link rel="canonical" href="index.html">
    <meta property="og:url" content="index.html">
  </head>
  <body class="u-body"><header class="u-clearfix u-header u-header" id="sec-9ffa"><div class="u-clearfix u-sheet u-sheet-1">
        <a href="Home.html" class="u-image u-logo u-image-1" data-image-width="183" data-image-height="83">
          <img src="images/0-Logo_from_FreeLogoDesign.png" class="u-logo-image u-logo-image-1" data-image-width="183">
        </a>
        <nav class="u-menu u-menu-dropdown u-offcanvas u-menu-1">
          <div class="menu-collapse" style="font-size: 1rem; letter-spacing: 0px;">
            <a class="u-button-style u-custom-left-right-menu-spacing u-custom-padding-bottom u-custom-text-hover-color u-custom-top-bottom-menu-spacing u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="#">
              <svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#menu-hamburger"></use></svg>
              <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><symbol id="menu-hamburger" viewBox="0 0 16 16" style="width: 16px; height: 16px;"><rect y="1" width="16" height="2"></rect><rect y="7" width="16" height="2"></rect><rect y="13" width="16" height="2"></rect>
</symbol>
</defs></svg>
            </a>
          </div>
          <div class="u-custom-menu u-nav-container">
            <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-3-base" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
          </div>
          <div class="u-custom-menu u-nav-container-collapse">
            <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
              <div class="u-sidenav-overflow">
                <div class="u-menu-close"></div>
                <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Home.html" style="padding: 16px 108px;">Home</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="Table-of-Contents.html" style="padding: 16px 108px;">Table of Contents</a>
</li><li class="u-nav-item"><a class="u-button-style u-nav-link" href="About-the-Author.html" style="padding: 16px 108px;">About the Author</a>
</li></ul>
              </div>
            </div>
            <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
          </div>
        </nav>
      </div></header> 
    <section class="u-clearfix u-valign-middle u-section-1" src="" id="sec-3f4f">
      <div class="u-expanded-height-lg u-expanded-height-md u-shape u-shape-svg u-text-palette-1-base u-shape-1">
        <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 100" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-ffba" style=""></use></svg>
        <svg class="u-svg-content" id="svg-ffba"><path d="M100,0l60,100H0V0H100z"></path></svg>
      </div>
      <div class="u-clearfix u-layout-wrap u-layout-wrap-1">
        <div class="u-layout">
          <div class="u-layout-row">
            <div class="u-align-center u-container-style u-layout-cell u-left-cell u-size-25 u-layout-cell-1">
              <div class="u-container-layout u-container-layout-1">
                <div alt="" class="u-image u-image-circle u-image-1" data-image-width="1920" data-image-height="1280"></div>
              </div>
            </div>
            <div class="u-container-style u-layout-cell u-right-cell u-shape-rectangle u-size-35 u-layout-cell-2">
              <div class="u-container-layout u-container-layout-2">
                <h1 class="u-align-left u-custom-font u-text u-text-font u-title u-text-1">Variants</h1>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-2" id="carousel_889b">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout">
            <div class="u-layout-row">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Single Nucleotide Polymorphism (SNP)</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">The genome, as
well as specific sequences, of one organism is not <i>exactly</i> the same as
the other. Generally, there is a consensus to what nucleotides should be in which
positions, which is based upon population genetics, that does produce a
reference genome for the organism. However, mutations do naturally occur and
this leads to variation in genomes of one organism to the next. These mutations
are polymorphisms in DNA sequences, which are determined as such when the
possible sequence variation is present in <i>at least 1% of the population</i>.
Single-point mutation are referred to as <i>single nucleotide polymorphism </i>(SNP)
whereas any mutations beyond a single point is <i>multiple nucleotide polymorphisms</i> (MNP). MNPs that are a result of genomic duplications are called <i>copy number
variation</i> (CNV). These mutations may be nonfunctional and silent by hiding
in the genomic sequence, or they may have detrimental consequences. Some functional
consequences of variants, especially SNPs, include changes to the amino acid
sequence, changes in the coding area that affect splicing, changes promoters/regulatory
regions that affect gene expression, changes that cause genetic disease, and changes
that increase medical risk or differential drug response.&nbsp;<br>
                    <br>
                    <br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-3" id="carousel_934b">
      <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
      <div class="u-clearfix u-gutter-30 u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-row">
            <div class="u-size-60">
              <div class="u-layout-col">
                <div class="u-align-justify u-container-style u-layout-cell u-right-cell u-size-60 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <p class="u-text u-text-1">SNPs can be divided into two broad categories based on its location and consequence: either protein-coding or not protein-coding. Protein-coding SNPs can be synonymous, where the SNP in the coding region&nbsp;<b>does not change</b>&nbsp;the amino acid sequence. This is due to the fact that codons are composed of triplet nucleotides and there are multiple codon combinations that can code for the same amino acid. Protein-coding SNPs can also be nonsynonymous, where the SNP in the coding region&nbsp;<b>changes</b>&nbsp;the amino acid sequence. Lastly, SNPs not in the coding region are broadly categorized as&nbsp;<i>noncoding SNPs</i>, which may compose of SNPs within introns and the UTRs. While noncoding SNPs are not seen, they can have prominent effects on the protein product by altering gene expression or gene regulation.&nbsp; &nbsp;<br>
                    </p>
                    <div class="u-shape u-shape-svg u-text-palette-1-base u-shape-2">
                      <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 130" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-5496"></use></svg>
                      <svg class="u-svg-content" id="svg-5496" style="enable-background:new 0 0 160 130;"><path d="M148.3,0H11.7C5.3,0,0,5.2,0,11.5v79.8c0,6.4,5.3,11.5,11.7,11.5h56L67.6,130l24.2-27.2h56.5c6.5,0,11.7-5.2,11.7-11.5V11.5
	C160,5.2,154.7,0,148.3,0z"></path></svg>
                    </div>
                    <h6 class="u-text u-text-white u-text-2">
                      <span style="font-size: 1.5rem;"><b>Practice Time!<br></b>
                      </span>
                      <br>Please watch these
two tutorials (<a href="https://new.galaxyproject.org/tutorials/g101/" class="u-active-none u-border-2 u-border-active-palette-2-dark-1 u-border-hover-palette-2-base u-border-white u-btn u-button-link u-button-style u-hover-none u-none u-text-hover-palette-2-base u-text-white u-btn-1">tutorial
1</a> and <a href="https://vimeo.com/75589889" class="u-active-none u-border-2 u-border-active-palette-2-dark-1 u-border-hover-palette-2-base u-border-white u-btn u-button-link u-button-style u-hover-none u-none u-text-body-alt-color u-text-hover-palette-2-base u-btn-2">tutorial 2</a>) on how to use
the Galaxy server’s tools to find SNPs using the dbSNP database. Note that as bioinformatic
tools are improved upon, the video may be slightly outdated in the version
numbers used. As an added bonus, click <a href="external/Unit%2006%20Reading%20-%20Viewing%20SNPs.pdf" class="u-active-none u-border-2 u-border-active-palette-2-dark-1 u-border-hover-palette-2-base u-border-white u-btn u-button-link u-button-style u-hover-none u-none u-text-body-alt-color u-text-hover-palette-2-base u-btn-3">here</a> for a step-by-step process on how to view SNP data with the genome browser IGV.
Since genome browsers are covered in a separate section, this may help connect
the dots. Note that the guide is based on the human hg18 assembly, which is not
the most up-to-date anymore.&nbsp;<i></i>
                      <br>
                    </h6>
                    <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-3"></div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-4" id="carousel_f1cf">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Variant Resources</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">Some database
resources for variants include <a href="https://www.ncbi.nlm.nih.gov/projects/SNP/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">dbSNP</a>, <a href="http://snpeffect.switchlab.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">SNPeffect</a>, <a href="http://www.snpedia.com/index.php/SNPedia" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">SNPedia</a>, <a href="http://www.ncbi.nlm.nih.gov/variation/tools/1000genomes/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">1000 Genomes
Browser</a>, and <a href="http://www.ncbi.nlm.nih.gov/clinvar/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">ClinVar</a>. The
database dbSNP, hosted by NCBI, is the largest database of SNPs that include a
primary database of submissions and a curated database of reference SNPs. In
addition to information on SNPs, it includes MNPs and short indels (insertion
or deletions). The SNPeffect database focuses on the effect of SNPs with their
proteins. It includes tools for protein folding and stability, with a predictor
for chaperone effects. The SNPedia database is a <i>Wikipedia</i>-based collection
of publications on known SNPs and their clinical significances. The 1000
Genomes Browser is a browser portal to the data collected from the 1000 Genomes
Project, which launched an extensive study in <a href="https://www.nature.com/articles/nature09534" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-6">2010</a> that utilized
sequence data from individuals in various geographical areas to find over
12,000 SNPs that included novel and/or rare SNPs that had an allele frequency of
less than 1% in the population. The ClinVar database is a curated database only
for human variations that have possible medical or clinical significance. The
database contains information from dbSNP, but should not replace dbSNP as a
resource. As ClinVar is directed towards medically-related research on variants,
the search functions allow the use of tags to query terms based on categories,
such as disease ([dis]) or genes ([gene]). Additional filters based on categories
applied on the variant’s records can be utilized on queried results to narrow
down the search.&nbsp;
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-5" id="carousel_a829">
      <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
      <div class="u-clearfix u-gutter-30 u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-row">
            <div class="u-size-60">
              <div class="u-layout-col">
                <div class="u-align-justify u-container-style u-layout-cell u-right-cell u-size-60 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <p class="u-text u-text-1">The extent of ClinVar’s ability as a database and connectivity with external resources is too broad to write here. Simultaneously, reading does not help the student learn everything. The best option is by performing- so let’s practice!&nbsp;&nbsp;<br>
                    </p>
                    <div class="u-shape u-shape-svg u-text-palette-1-base u-shape-2">
                      <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 130" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-5496"></use></svg>
                      <svg class="u-svg-content" id="svg-5496" style="enable-background:new 0 0 160 130;"><path d="M148.3,0H11.7C5.3,0,0,5.2,0,11.5v79.8c0,6.4,5.3,11.5,11.7,11.5h56L67.6,130l24.2-27.2h56.5c6.5,0,11.7-5.2,11.7-11.5V11.5
	C160,5.2,154.7,0,148.3,0z"></path></svg>
                    </div>
                    <h6 class="u-text u-text-white u-text-2">
                      <span style="font-size: 1.5rem;"><b>Practice Time!<br></b>
                      </span>
                      <br>Find
the human CACNA1A gene in NCBI ClinVar. Filter to limit results to Variation
type: Deletion. How many total variations are listed after filtering? Your
results should look similar to the top figure. Now, apply the filter for <i>Pathogenic</i>.
How many variants from are pathogenic? Your results should look similar to the
bottom figure. Note that as the database grows and more information is added,
the total number will be changed!<br>
                    </h6>
                    <img src="images/7-Variants-ClinVar1_NCBI.png" alt="" class="u-image u-image-default u-image-1" data-image-width="1458" data-image-height="692">
                    <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-3"></div>
                    <img src="images/7-Variants-ClinVar2_NCBI.png" alt="" class="u-image u-image-default u-image-2" data-image-width="1458" data-image-height="692">
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-6" id="carousel_5788">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-clearfix u-expanded-width u-layout-wrap u-layout-wrap-1">
          <div class="u-layout" style="">
            <div class="u-layout-row" style="">
              <div class="u-container-style u-layout-cell u-size-60 u-layout-cell-1">
                <div class="u-container-layout u-container-layout-1">
                  <h2 class="u-align-left u-text u-text-1">Copy Number Variation (CNV)</h2>
                  <div class="u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
                  <p class="u-align-justify u-text u-text-2">As previously mentioned, mutations can include duplications of genomic regions, resulting in CNVs. Individuals within a population can vary in the number of times the genomic region of the variation is present in their genome.<b>&nbsp;</b>CNVs have been found in the genomes of humans, flies, mice, monkey, and chimpanzees. CNVs were first found in 2004 by&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pubmed/15273396" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">Sebat et al.</a>&nbsp;with a more comprehensive study done in 2007 by&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pubmed/17160897" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">Wong et al.</a>&nbsp;that showed CNVs overlapping with genes and miRNA. Many of the overlapped genes were sensory-related, suggesting that CNVs may contribute to differences in taste or smell between populations. The genesis of CNVs is still elusive, but studies were performed to investigate their origin. A study by&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pubmed/19597530" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">Hastings et al.</a>&nbsp;suggested the microhomology-mediated break induced replication (MMBIR) model for explaining the formation of new CNVs based on sequence and models for DNA repair. Further studies found CNV variation in inbred mice, which suggesting spontaneous development<b>.&nbsp;</b>Links to genetic disorder have been found to be associated with some CNVs. A study by&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pubmed/18304490" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-4">Bruder et al.</a>&nbsp;found that identical twins have differences in CNV.&nbsp;<br>
                    <br>As new research
is being performed on CNVs, the databases related to CNVs are flooded with new
information. Some databases may not have the up-to-date information depending
on how well it is maintained. The<b></b>Database of Genomic Variants (<a href="http://dgv.tcag.ca/dgv/app/home" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-5">DGV</a>) database contains a genome
browser that shows CNV gains in blue and CNV losses in red in datasets, which
is also available as the UCSC Genome Browser track <i>DGV Struct Var</i>. The Database
of Chromosomal Imbalance and Phenotype in Humans using Ensembl Resources (<a href="http://decipher.sanger.ac.uk/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-6">DECIPHER</a>) database is built on patient
data, obtained with consent. Lastly, the <a href="http://www.ncbi.nlm.nih.gov/dbvar/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-7">dbVar</a> database from NCBI uses
data from various databases, including DGV. Another database resource includes
the Broad Institute’s The Cancer Genome Analysis (TCGA) group that hosts the
analysis structure <a href="http://www.broadinstitute.org/cancer/cga/Firehose" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-8">Firehose</a>.
Firehose contains data from different forms of cancer and, within each cancer
type, has a CNV option to view a patient’s data with relation to loss of CNV (blue)
or gain of CNV (red). It also includes a CNV summary track that shows a general
summary of results between a red and blue split. The NCBI <a href="https://www.ncbi.nlm.nih.gov/variation/view" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-9">Variation Viewer</a> contains
data from NCBI’s dbSNP and dbVar databases in the form of tracks. Lastly, the <a href="https://gnomad.broadinstitute.org/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-10">GnomAD</a> database contains a searchable
browser to show peaks for the quantity of protein-coding SNPs found in a gene
region with CNV data underneath the peaks that are color coded for gain (brown)
and loss (red) of CNV.&nbsp; &nbsp; &nbsp;&nbsp;<br>
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <section class="u-clearfix u-section-7" id="carousel_a5ab">
      <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-1"></div>
      <div class="u-clearfix u-gutter-30 u-layout-wrap u-layout-wrap-1">
        <div class="u-gutter-0 u-layout">
          <div class="u-layout-row">
            <div class="u-size-60">
              <div class="u-layout-col">
                <div class="u-align-justify u-container-style u-layout-cell u-right-cell u-size-60 u-layout-cell-1">
                  <div class="u-container-layout u-container-layout-1">
                    <div class="u-opacity u-opacity-10 u-palette-1-base u-shape u-shape-rectangle u-shape-2"></div>
                    <p class="u-text u-text-1">Other resources
available for CNV include Bioconductor packages for use with R. The<i>
                        <a href="http://www.bioconductor.org/packages/release/bioc/html/cn.mops.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-1">cn.MOPS</a></i>package analyzes next-generation sequencing data to find CNVs in
individuals. This package was described by <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3351174/" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-2">Klambauer et al.</a> in addition to using two methods to detect CNVs: read count detection and
reference-based approach. The <i>
                        <a href="http://www.bioconductor.org/packages/release/bioc/html/CNVtools.html" class="u-active-none u-border-none u-btn u-button-style u-hover-none u-none u-text-palette-1-base u-btn-3">CNVtools</a></i> package looks for genetic associations in cases versus controls.&nbsp;<b></b>
                      <br>
							 <br>
                    </p>
                    <div class="u-shape u-shape-svg u-text-palette-1-base u-shape-3">
                      <svg class="u-svg-link" preserveAspectRatio="none" viewBox="0 0 160 130" style=""><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-5496"></use></svg>
                      <svg class="u-svg-content" id="svg-5496" style="enable-background:new 0 0 160 130;"><path d="M148.3,0H11.7C5.3,0,0,5.2,0,11.5v79.8c0,6.4,5.3,11.5,11.7,11.5h56L67.6,130l24.2-27.2h56.5c6.5,0,11.7-5.2,11.7-11.5V11.5
	C160,5.2,154.7,0,148.3,0z"></path></svg>
                    </div>
                    <h6 class="u-text u-text-white u-text-2">
                      <span style="font-size: 1.5rem;"><b>Practice Time!<br></b>
                      </span>
                      <br>Find
the CACNA1A gene in the NCBI Variation Viewer. Filter for dbVar and CNV. How
many total CNVs? Your results should look similar to the figure below. Note
that as the database grows and more information is added, the total number will
be changed!<br>
                    </h6>
                    <img src="images/7-Variants-VariationViewer_NCBI.png" alt="" class="u-image u-image-default u-image-1" data-image-width="1440" data-image-height="424">
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    
	 <section class="u-clearfix u-section-9" id="sec-d940">
      <div class="u-clearfix u-sheet u-sheet-1">
        <div class="u-border-9 u-border-palette-1-light-2 u-container-style u-expanded-width u-group u-radius-50 u-shape-round u-group-1">
          <div class="u-container-layout u-container-layout-1">
            <h3 class="u-align-center u-text u-text-1">Variants</h3>
            <a href="DatabasesGenomeBrowser.html" class="u-btn u-btn-round u-button-style u-hover-palette-4-base u-palette-1-base u-radius-2 u-btn-1">Next Unit&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 492.004 492.004" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M484.14,226.886L306.46,49.202c-5.072-5.072-11.832-7.856-19.04-7.856c-7.216,0-13.972,2.788-19.044,7.856l-16.132,16.136    c-5.068,5.064-7.86,11.828-7.86,19.04c0,7.208,2.792,14.2,7.86,19.264L355.9,207.526H26.58C11.732,207.526,0,219.15,0,234.002    v22.812c0,14.852,11.732,27.648,26.58,27.648h330.496L252.248,388.926c-5.068,5.072-7.86,11.652-7.86,18.864    c0,7.204,2.792,13.88,7.86,18.948l16.132,16.084c5.072,5.072,11.828,7.836,19.044,7.836c7.208,0,13.968-2.8,19.04-7.872    l177.68-177.68c5.084-5.088,7.88-11.88,7.86-19.1C492.02,238.762,489.228,231.966,484.14,226.886z"></path>
</g>
</g></svg><img></span>
            </a>
            <a href="RNA-and-Ultraconserved-Regions.html" class="u-btn u-btn-round u-button-style u-hover-palette-1-light-2 u-palette-1-base u-radius-2 u-text-hover-palette-4-base u-btn-2">&nbsp;<span class="u-icon"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" xml:space="preserve" class="u-svg-content" viewBox="0 0 447.243 447.243" x="0px" y="0px" style="width: 1em; height: 1em;"><g><g><path d="M420.361,192.229c-1.83-0.297-3.682-0.434-5.535-0.41H99.305l6.88-3.2c6.725-3.183,12.843-7.515,18.08-12.8l88.48-88.48    c11.653-11.124,13.611-29.019,4.64-42.4c-10.441-14.259-30.464-17.355-44.724-6.914c-1.152,0.844-2.247,1.764-3.276,2.754    l-160,160C-3.119,213.269-3.13,233.53,9.36,246.034c0.008,0.008,0.017,0.017,0.025,0.025l160,160    c12.514,12.479,32.775,12.451,45.255-0.063c0.982-0.985,1.899-2.033,2.745-3.137c8.971-13.381,7.013-31.276-4.64-42.4    l-88.32-88.64c-4.695-4.7-10.093-8.641-16-11.68l-9.6-4.32h314.24c16.347,0.607,30.689-10.812,33.76-26.88    C449.654,211.494,437.806,195.059,420.361,192.229z"></path>
</g>
</g></svg><img></span> &nbsp;Previous Unit
            </a>
          </div>
        </div>
      </div>
    </section>
    
    <footer class="u-align-center u-clearfix u-footer u-grey-80" id="sec-0012"><div class="u-clearfix u-sheet u-sheet-1"></div></footer>
    <section class="u-backlink u-clearfix u-grey-80">
           
      <p class="u-text">This webpage was produced by Brian Poon as the Final Porfolio for class AS.410.610.82, &nbsp; <br>Bioinformatics: Tools for Genome Analysis</i>, at Johns Hopkins University.&nbsp;
      </p>
      
    </section>
  </body>
</html>